<!DOCTYPE html>
<!-- saved from url=(0064)https://github.com/griffithlab/rnaseq_tutorial/wiki/IGV-Tutorial -->
<html lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
    
  <link rel="dns-prefetch" href="https://assets-cdn.github.com/">
  <link rel="dns-prefetch" href="https://avatars0.githubusercontent.com/">
  <link rel="dns-prefetch" href="https://avatars1.githubusercontent.com/">
  <link rel="dns-prefetch" href="https://avatars2.githubusercontent.com/">
  <link rel="dns-prefetch" href="https://avatars3.githubusercontent.com/">
  <link rel="dns-prefetch" href="https://github-cloud.s3.amazonaws.com/">
  <link rel="dns-prefetch" href="https://user-images.githubusercontent.com/">



  <link crossorigin="anonymous" media="all" integrity="sha512-PkbtxdWDpLChpxtWQ0KbvJoef4XMYPq5pfd/ZmylYZTzXYpCfGwN9d+bsSKcmOJLwTkfjFkfj5wz3poDrhJoSQ==" rel="stylesheet" href="./IGV Tutorial · griffithlab_rnaseq_tutorial Wiki_files/frameworks-f6e6ce21346c0d2eb22def1e8534afcb.css">
  <link crossorigin="anonymous" media="all" integrity="sha512-EuadCD9/LFVfed/klpP7kj+qTP0hWhyJ9KN6kLBEXRSLjXEKjm4Z4JORqgppbiE/toxjWa60Fjsh9BtIJdLq9g==" rel="stylesheet" href="./IGV Tutorial · griffithlab_rnaseq_tutorial Wiki_files/github-6353b64247f8dd959d2b123ae98fba98.css">
  
  
  
  

  <meta name="viewport" content="width=device-width">
  
  <title>IGV Tutorial · griffithlab/rnaseq_tutorial Wiki</title>
    <meta name="description" content="GitHub is where people build software. More than 28 million people use GitHub to discover, fork, and contribute to over 85 million projects.">
    <link rel="search" type="application/opensearchdescription+xml" href="https://github.com/opensearch.xml" title="GitHub">
  <link rel="fluid-icon" href="https://github.com/fluidicon.png" title="GitHub">
  <meta property="fb:app_id" content="1401488693436528">

    
    <meta property="og:image" content="https://avatars3.githubusercontent.com/u/8323084?s=400&amp;v=4"><meta property="og:site_name" content="GitHub"><meta property="og:type" content="object"><meta property="og:title" content="griffithlab/rnaseq_tutorial"><meta property="og:url" content="https://github.com/griffithlab/rnaseq_tutorial"><meta property="og:description" content="rnaseq_tutorial - Informatics for RNA-seq: A web resource for analysis on the cloud. Educational tutorials and working pipelines for RNA-seq analysis including an introduction to: cloud computing, ...">

  <link rel="assets" href="https://assets-cdn.github.com/">
  <link rel="web-socket" href="wss://live.github.com/_sockets/VjI6MjgzMjcwNTkzOjk3ZWNhYWMyMmM1ODYyOGJkMmJiZGYxYWMwZWQwZTk2NTZkOTU3N2VlZjIzYmZlMTc3MzZkNmM1ZDUyNTE3Y2E=--4b844e70dc3a64bda07306e3c77f58152404e87a">
  <meta name="pjax-timeout" content="1000">
  <link rel="sudo-modal" href="https://github.com/sessions/sudo_modal">
  <meta name="request-id" content="D43B:6023:20683E:3F21EB:5B1ED9CD" data-pjax-transient="">


  

  <meta name="selected-link" value="repo_wiki" data-pjax-transient="">

    <meta name="google-site-verification" content="KT5gs8h0wvaagLKAVWq8bbeNwnZZK1r1XQysX3xurLU">
  <meta name="google-site-verification" content="ZzhVyEFwb7w3e0-uOTltm8Jsck2F5StVihD0exw2fsA">
  <meta name="google-site-verification" content="GXs5KoUUkNCoaAZn7wPN-t01Pywp9M3sEjnt_3_ZWPc">
    <meta name="google-analytics" content="UA-3769691-2">

<meta name="octolytics-host" content="collector.githubapp.com"><meta name="octolytics-app-id" content="github"><meta name="octolytics-event-url" content="https://collector.githubapp.com/github-external/browser_event"><meta name="octolytics-dimension-request_id" content="D43B:6023:20683E:3F21EB:5B1ED9CD"><meta name="octolytics-dimension-region_edge" content="iad"><meta name="octolytics-dimension-region_render" content="iad"><meta name="octolytics-actor-id" content="39917801"><meta name="octolytics-actor-login" content="MalikSediqzad"><meta name="octolytics-actor-hash" content="3a77115565b5b3f0d258c31a47ab9d3574bed0fe8d1947ba151024893d1ed506">
<meta name="analytics-location" content="/&lt;user-name&gt;/&lt;repo-name&gt;/wiki/show" data-pjax-transient="true">




  <meta class="js-ga-set" name="dimension1" content="Logged In">


  

      <meta name="hostname" content="github.com">
    <meta name="user-login" content="MalikSediqzad">

      <meta name="expected-hostname" content="github.com">
    <meta name="js-proxy-site-detection-payload" content="NjM3YTEwZWM4MDdiMTQ3NTFiMzMyMWM3ZTM5YzY0MzEyODNmODUyNjViMDRjOTg0MWFlMTNkYjZhNThiN2I5NXx7InJlbW90ZV9hZGRyZXNzIjoiMTI4LjI1Mi4yMzMuMjQ0IiwicmVxdWVzdF9pZCI6IkQ0M0I6NjAyMzoyMDY4M0U6M0YyMUVCOjVCMUVEOUNEIiwidGltZXN0YW1wIjoxNTI4NzQ4NDkzLCJob3N0IjoiZ2l0aHViLmNvbSJ9">

    <meta name="enabled-features" content="UNIVERSE_BANNER,FREE_TRIALS,MARKETPLACE_INSIGHTS,MARKETPLACE_INSIGHTS_CONVERSION_PERCENTAGES">

  <meta name="html-safe-nonce" content="a991ce52d7e3d94962cea063af36b4807faeb9e8">

  <meta http-equiv="x-pjax-version" content="20d9bec307536519e5b0bc7ea8bbab87">
  

      <link href="https://github.com/griffithlab/rnaseq_tutorial/commits/master.atom" rel="alternate" title="Recent Commits to rnaseq_tutorial:master" type="application/atom+xml">

  <meta name="description" content="rnaseq_tutorial - Informatics for RNA-seq: A web resource for analysis on the cloud. Educational tutorials and working pipelines for RNA-seq analysis including an introduction to: cloud computing, critical file formats, reference genomes, gene annotation, expression, differential expression, alternative splicing, data visualization, and interpretation.">
  <meta name="go-import" content="github.com/griffithlab/rnaseq_tutorial git https://github.com/griffithlab/rnaseq_tutorial.git">

  <meta name="octolytics-dimension-user_id" content="8323084"><meta name="octolytics-dimension-user_login" content="griffithlab"><meta name="octolytics-dimension-repository_id" content="22481024"><meta name="octolytics-dimension-repository_nwo" content="griffithlab/rnaseq_tutorial"><meta name="octolytics-dimension-repository_public" content="true"><meta name="octolytics-dimension-repository_is_fork" content="false"><meta name="octolytics-dimension-repository_network_root_id" content="22481024"><meta name="octolytics-dimension-repository_network_root_nwo" content="griffithlab/rnaseq_tutorial"><meta name="octolytics-dimension-repository_explore_github_marketplace_ci_cta_shown" content="false">




  <meta name="browser-stats-url" content="https://api.github.com/_private/browser/stats">

  <meta name="browser-errors-url" content="https://api.github.com/_private/browser/errors">

  <link rel="mask-icon" href="https://assets-cdn.github.com/pinned-octocat.svg" color="#000000">
  <link rel="icon" type="image/x-icon" class="js-site-favicon" href="https://assets-cdn.github.com/favicon.ico">

<meta name="theme-color" content="#1e2327">


  <meta name="u2f-support" content="true">

<link rel="manifest" href="https://github.com/manifest.json" crossorigin="use-credentials">

  </head>

  <body class="logged-in env-production emoji-size-boost intent-mouse">
    

  <div class="position-relative js-header-wrapper ">
    <a href="https://github.com/griffithlab/rnaseq_tutorial/wiki/IGV-Tutorial#start-of-content" tabindex="1" class="p-3 bg-blue text-white show-on-focus js-skip-to-content">Skip to content</a>
    <div id="js-pjax-loader-bar" class="pjax-loader-bar"><div class="progress"></div></div>

    
    
    



        
<header class="Header  f5" role="banner">
  <div class="d-flex flex-justify-between px-3 container-lg">
    <div class="d-flex flex-justify-between ">
      <div class="">
        <a class="header-logo-invertocat" href="https://github.com/" data-hotkey="g d" aria-label="Homepage" data-ga-click="Header, go to dashboard, icon:logo">
  <svg height="32" class="octicon octicon-mark-github" viewBox="0 0 16 16" version="1.1" width="32" aria-hidden="true"><path fill-rule="evenodd" d="M8 0C3.58 0 0 3.58 0 8c0 3.54 2.29 6.53 5.47 7.59.4.07.55-.17.55-.38 0-.19-.01-.82-.01-1.49-2.01.37-2.53-.49-2.69-.94-.09-.23-.48-.94-.82-1.13-.28-.15-.68-.52-.01-.53.63-.01 1.08.58 1.23.82.72 1.21 1.87.87 2.33.66.07-.52.28-.87.51-1.07-1.78-.2-3.64-.89-3.64-3.95 0-.87.31-1.59.82-2.15-.08-.2-.36-1.02.08-2.12 0 0 .67-.21 2.2.82.64-.18 1.32-.27 2-.27.68 0 1.36.09 2 .27 1.53-1.04 2.2-.82 2.2-.82.44 1.1.16 1.92.08 2.12.51.56.82 1.27.82 2.15 0 3.07-1.87 3.75-3.65 3.95.29.25.54.73.54 1.48 0 1.07-.01 1.93-.01 2.2 0 .21.15.46.55.38A8.013 8.013 0 0 0 16 8c0-4.42-3.58-8-8-8z"></path></svg>
</a>

      </div>

    </div>

    <div class="HeaderMenu d-flex flex-justify-between flex-auto">
      <div class="d-flex">
            <div class="">
              <div class="header-search scoped-search site-scoped-search js-site-search position-relative" role="search">
  <div class="position-relative">
    <!-- '"` --><!-- </textarea></xmp> --><form class="js-site-search-form" data-scope-type="Repository" data-scope-id="22481024" data-scoped-search-url="/griffithlab/rnaseq_tutorial/search" data-unscoped-search-url="/search" action="https://github.com/griffithlab/rnaseq_tutorial/search" accept-charset="UTF-8" method="get"><input name="utf8" type="hidden" value="✓">
      <label class="form-control header-search-wrapper header-search-wrapper-jump-to position-relative d-flex flex-justify-between flex-items-center js-chromeless-input-container">
        <input type="text" class="form-control header-search-input jump-to-field js-jump-to-field js-site-search-focus js-site-search-field is-clearable" data-hotkey="s,/" name="q" value="" placeholder="Search or jump to…" aria-label="Search this repository" data-unscoped-placeholder="Search or jump to…" data-scoped-placeholder="Search or jump to…" data-jump-to-suggestions-path="/_graphql/GetSuggestedNavigationDestinations#csrf-token=wHZGvL3IUIBo+iC5elhDvu3MsCgId+cNYD/KV0FgS5OFrpwsLqf1Np9kZfM77+r4iiran33oRYYzoldQi91icw==" spellcheck="false" autocomplete="off" autocapitalize="off">
          <input type="hidden" class="js-site-search-type-field" name="type">
            <img src="./IGV Tutorial · griffithlab_rnaseq_tutorial Wiki_files/search-shortcut-hint.svg" alt="" class="mr-2 header-search-key-slash">

            <div class="Box position-absolute overflow-hidden d-none jump-to-suggestions js-jump-to-suggestions-container">
              <ul class="d-none js-jump-to-suggestions-template-container">
                <li class="d-flex flex-justify-start flex-items-center p-0 f5 navigation-item js-navigation-item">
                  <a class="no-underline d-flex flex-auto flex-items-center p-2 jump-to-suggestions-path js-jump-to-suggestion-path js-navigation-open" aria-label="Jump to..." href="https://github.com/griffithlab/rnaseq_tutorial/wiki/IGV-Tutorial">
                    <div class="jump-to-octicon js-jump-to-octicon mr-2 text-center d-none"></div>
                    <img class="avatar mr-2 flex-shrink-0 js-jump-to-suggestion-avatar" alt="" src="https://github.com/griffithlab/rnaseq_tutorial/wiki/IGV-Tutorial" width="28" height="28">
                    <div class="jump-to-suggestion-name js-jump-to-suggestion-name flex-auto overflow-hidden no-wrap css-truncate css-truncate-target">
                    </div>

                    <div class="border rounded-1 flex-shrink-0 bg-gray px-1 text-gray-light ml-1 f6 d-none js-jump-to-badge-search">
                      <span class="js-jump-to-badge-search-text-default d-none">
                        In this repository
                      </span>
                      <span class="js-jump-to-badge-search-text-global d-none">
                        All GitHub
                      </span>
                      <span class="d-inline-block ml-1 v-align-middle">↵</span>
                    </div>

                    <div class="border rounded-1 flex-shrink-0 bg-gray px-1 text-gray-light ml-1 f6 d-none d-on-nav-focus js-jump-to-badge-jump">
                      Jump to
                      <span class="d-inline-block ml-1 v-align-middle">↵</span>
                    </div>
                  </a>
                </li>
                <svg height="16" width="16" class="octicon octicon-repo flex-shrink-0 js-jump-to-repo-octicon-template" title="Repository" viewBox="0 0 12 16" version="1.1" aria-hidden="true"><path fill-rule="evenodd" d="M4 9H3V8h1v1zm0-3H3v1h1V6zm0-2H3v1h1V4zm0-2H3v1h1V2zm8-1v12c0 .55-.45 1-1 1H6v2l-1.5-1.5L3 16v-2H1c-.55 0-1-.45-1-1V1c0-.55.45-1 1-1h10c.55 0 1 .45 1 1zm-1 10H1v2h2v-1h3v1h5v-2zm0-10H2v9h9V1z"></path></svg>
                <svg height="16" width="16" class="octicon octicon-project flex-shrink-0 js-jump-to-project-octicon-template" title="Project" viewBox="0 0 15 16" version="1.1" aria-hidden="true"><path fill-rule="evenodd" d="M10 12h3V2h-3v10zm-4-2h3V2H6v8zm-4 4h3V2H2v12zm-1 1h13V1H1v14zM14 0H1a1 1 0 0 0-1 1v14a1 1 0 0 0 1 1h13a1 1 0 0 0 1-1V1a1 1 0 0 0-1-1z"></path></svg>
                <svg height="16" width="16" class="octicon octicon-search flex-shrink-0 js-jump-to-search-octicon-template" title="Search" viewBox="0 0 16 16" version="1.1" aria-hidden="true"><path fill-rule="evenodd" d="M15.7 13.3l-3.81-3.83A5.93 5.93 0 0 0 13 6c0-3.31-2.69-6-6-6S1 2.69 1 6s2.69 6 6 6c1.3 0 2.48-.41 3.47-1.11l3.83 3.81c.19.2.45.3.7.3.25 0 .52-.09.7-.3a.996.996 0 0 0 0-1.41v.01zM7 10.7c-2.59 0-4.7-2.11-4.7-4.7 0-2.59 2.11-4.7 4.7-4.7 2.59 0 4.7 2.11 4.7 4.7 0 2.59-2.11 4.7-4.7 4.7z"></path></svg>
              </ul>
              <ul class="d-none js-jump-to-no-results-template-container">
                <li class="d-flex flex-justify-center flex-items-center p-3 f5 d-none">
                  <span class="text-gray">No suggested jump to results</span>
                </li> 
              </ul>

              <ul class="js-navigation-container jump-to-suggestions-results-container js-jump-to-suggestions-results-container">
                <li class="d-flex flex-justify-center flex-items-center p-0 f5">
                  <img src="./IGV Tutorial · griffithlab_rnaseq_tutorial Wiki_files/octocat-spinner-128.gif" alt="Octocat Spinner Icon" class="m-2" width="28">
                </li>
              </ul>
            </div>
      </label>
</form>  </div>
</div>

            </div>

          <ul class="d-flex pl-2 flex-items-center text-bold list-style-none" role="navigation">
            <li>
              <a class="js-selected-navigation-item HeaderNavlink px-2" data-hotkey="g p" data-ga-click="Header, click, Nav menu - item:pulls context:user" aria-label="Pull requests you created" data-selected-links="/pulls /pulls/assigned /pulls/mentioned /pulls" href="https://github.com/pulls">
                Pull requests
</a>            </li>
            <li>
              <a class="js-selected-navigation-item HeaderNavlink px-2" data-hotkey="g i" data-ga-click="Header, click, Nav menu - item:issues context:user" aria-label="Issues you created" data-selected-links="/issues /issues/assigned /issues/mentioned /issues" href="https://github.com/issues">
                Issues
</a>            </li>
                <li>
                  <a class="js-selected-navigation-item HeaderNavlink px-2" data-ga-click="Header, click, Nav menu - item:marketplace context:user" data-octo-click="marketplace_click" data-octo-dimensions="location:nav_bar, group:apps_text" data-selected-links=" /marketplace" href="https://github.com/marketplace">
                    Apps
</a>                </li>
            <li>
              <a class="js-selected-navigation-item HeaderNavlink px-2" data-ga-click="Header, click, Nav menu - item:explore" data-selected-links="/explore /trending /trending/developers /integrations /integrations/feature/code /integrations/feature/collaborate /integrations/feature/ship showcases showcases_search showcases_landing /explore" href="https://github.com/explore">
                Explore
</a>            </li>
          </ul>
      </div>

      <div class="d-flex">
        
<ul class="user-nav d-flex flex-items-center list-style-none" id="user-links">
  <li class="dropdown">
    <span class="d-inline-block  px-2">
      
    <a aria-label="You have no unread notifications" class="notification-indicator tooltipped tooltipped-s  js-socket-channel js-notification-indicator" data-hotkey="g n" data-ga-click="Header, go to notifications, icon:read" data-channel="notification-changed:39917801" href="https://github.com/notifications">
        <span class="mail-status "></span>
        <svg class="octicon octicon-bell" viewBox="0 0 14 16" version="1.1" width="14" height="16" aria-hidden="true"><path fill-rule="evenodd" d="M13.99 11.991v1H0v-1l.73-.58c.769-.769.809-2.547 1.189-4.416.77-3.767 4.077-4.996 4.077-4.996 0-.55.45-1 .999-1 .55 0 1 .45 1 1 0 0 3.387 1.229 4.156 4.996.38 1.879.42 3.657 1.19 4.417l.659.58h-.01zM6.995 15.99c1.11 0 1.999-.89 1.999-1.999H4.996c0 1.11.89 1.999 1.999 1.999z"></path></svg>
</a>
    </span>
  </li>

  <li class="dropdown">
    <details class="details-expanded details-reset js-dropdown-details d-flex px-2 flex-items-center">
      <summary class="HeaderNavlink" aria-label="Create new…" data-ga-click="Header, create new, icon:add">
        <svg class="octicon octicon-plus float-left mr-1 mt-1" viewBox="0 0 12 16" version="1.1" width="12" height="16" aria-hidden="true"><path fill-rule="evenodd" d="M12 9H7v5H5V9H0V7h5V2h2v5h5v2z"></path></svg>
        <span class="dropdown-caret mt-1"></span>
      </summary>

      <ul class="dropdown-menu dropdown-menu-sw">
        
<a class="dropdown-item" href="https://github.com/new" data-ga-click="Header, create new repository">
  New repository
</a>

  <a class="dropdown-item" href="https://github.com/new/import" data-ga-click="Header, import a repository">
    Import repository
  </a>

<a class="dropdown-item" href="https://gist.github.com/" data-ga-click="Header, create new gist">
  New gist
</a>

  <a class="dropdown-item" href="https://github.com/organizations/new" data-ga-click="Header, create new organization">
    New organization
  </a>



  <div class="dropdown-divider"></div>
  <div class="dropdown-header">
    <span title="griffithlab/rnaseq_tutorial">This repository</span>
  </div>
    <a class="dropdown-item" href="https://github.com/griffithlab/rnaseq_tutorial/issues/new" data-ga-click="Header, create new issue">
      New issue
    </a>

      </ul>
    </details>
  </li>

  <li class="dropdown">

    <details class="details-expanded details-reset js-dropdown-details d-flex pl-2 flex-items-center">
      <summary class="HeaderNavlink name mt-1" aria-label="View profile and more" data-ga-click="Header, show menu, icon:avatar">
        <img alt="@MalikSediqzad" class="avatar float-left mr-1" src="./IGV Tutorial · griffithlab_rnaseq_tutorial Wiki_files/39917801" height="20" width="20">
        <span class="dropdown-caret"></span>
      </summary>

      <ul class="dropdown-menu dropdown-menu-sw">
        <li class="dropdown-header header-nav-current-user css-truncate">
          Signed in as <strong class="css-truncate-target">MalikSediqzad</strong>
        </li>

        <li class="dropdown-divider"></li>

        <li><a class="dropdown-item" href="https://github.com/MalikSediqzad" data-ga-click="Header, go to profile, text:your profile">
          Your profile
        </a></li>
        <li><a class="dropdown-item" href="https://github.com/MalikSediqzad?tab=stars" data-ga-click="Header, go to starred repos, text:your stars">
          Your stars
        </a></li>
          <li><a class="dropdown-item" href="https://gist.github.com/" data-ga-click="Header, your gists, text:your gists">Your gists</a></li>

        <li class="dropdown-divider"></li>

        <li><a class="dropdown-item" href="https://help.github.com/" data-ga-click="Header, go to help, text:help">
          Help
        </a></li>

        <li><a class="dropdown-item" href="https://github.com/settings/profile" data-ga-click="Header, go to settings, icon:settings">
          Settings
        </a></li>

        <li><!-- '"` --><!-- </textarea></xmp> --><form class="logout-form" action="https://github.com/logout" accept-charset="UTF-8" method="post"><input name="utf8" type="hidden" value="✓"><input type="hidden" name="authenticity_token" value="vnv0smD5mRZZmTukhYePXtO8jnFNMNgVLRi6g6+J3+iG+M+JfJ9XQ/7GjLEodzsr7wLJXsN+NWGMW4r0R7pyIA==">
          <button type="submit" class="dropdown-item dropdown-signout" data-ga-click="Header, sign out, icon:logout">
            Sign out
          </button>
        </form></li>
      </ul>
    </details>
  </li>
</ul>



        <!-- '"` --><!-- </textarea></xmp> --><form class="sr-only right-0" action="https://github.com/logout" accept-charset="UTF-8" method="post"><input name="utf8" type="hidden" value="✓"><input type="hidden" name="authenticity_token" value="Khw+1qFB8R8TpKtOaDyTs9Fx8DZF4ggC2UuVZUPU5rESnwXtvSc/SrT7HFvFzCfG7c+3Gcus5XZ4CKUSq+dLeQ==">
          <button type="submit" class="dropdown-item dropdown-signout" data-ga-click="Header, sign out, icon:logout">
            Sign out
          </button>
</form>      </div>
    </div>
  </div>
</header>

      

  </div>

  <div id="start-of-content" class="show-on-focus"></div>

    <div id="js-flash-container">
</div>



  <div role="main" class="application-main ">
        <div itemscope="" itemtype="http://schema.org/SoftwareSourceCode" class="">
    <div id="js-repo-pjax-container" data-pjax-container="">
      



  



  <div class="pagehead repohead instapaper_ignore readability-menu experiment-repo-nav  ">
    <div class="repohead-details-container clearfix container">

      <ul class="pagehead-actions">
  <li>
        <!-- '"` --><!-- </textarea></xmp> --><form data-autosubmit="true" data-remote="true" class="js-social-container" action="https://github.com/notifications/subscribe" accept-charset="UTF-8" method="post"><input name="utf8" type="hidden" value="✓"><input type="hidden" name="authenticity_token" value="1/PjhSR9+iR+zyFzrxbjp/e29hUWe8t/CGXRev8S1r+9FbR56r4JgJyNmq9gKlnw74V+7wlrwdS5JFjDglkGgg==">      <input type="hidden" name="repository_id" id="repository_id" value="22481024" class="form-control">

        <div class="select-menu js-menu-container js-select-menu">
          <a href="https://github.com/griffithlab/rnaseq_tutorial/subscription" class="btn btn-sm btn-with-count select-menu-button js-menu-target" role="button" aria-haspopup="true" aria-expanded="false" aria-label="Toggle repository notifications menu" data-ga-click="Repository, click Watch settings, action:wiki#show">
            <span class="js-select-button">
                <svg class="octicon octicon-eye" viewBox="0 0 16 16" version="1.1" width="16" height="16" aria-hidden="true"><path fill-rule="evenodd" d="M8.06 2C3 2 0 8 0 8s3 6 8.06 6C13 14 16 8 16 8s-3-6-7.94-6zM8 12c-2.2 0-4-1.78-4-4 0-2.2 1.8-4 4-4 2.22 0 4 1.8 4 4 0 2.22-1.78 4-4 4zm2-4c0 1.11-.89 2-2 2-1.11 0-2-.89-2-2 0-1.11.89-2 2-2 1.11 0 2 .89 2 2z"></path></svg>
                Watch
            </span>
          </a>
          <a class="social-count js-social-count" href="https://github.com/griffithlab/rnaseq_tutorial/watchers" aria-label="133 users are watching this repository">
            133
          </a>

        <div class="select-menu-modal-holder">
          <div class="select-menu-modal subscription-menu-modal js-menu-content">
            <div class="select-menu-header js-navigation-enable" tabindex="-1">
              <svg class="octicon octicon-x js-menu-close" role="img" aria-label="Close" viewBox="0 0 12 16" version="1.1" width="12" height="16"><path fill-rule="evenodd" d="M7.48 8l3.75 3.75-1.48 1.48L6 9.48l-3.75 3.75-1.48-1.48L4.52 8 .77 4.25l1.48-1.48L6 6.52l3.75-3.75 1.48 1.48L7.48 8z"></path></svg>
              <span class="select-menu-title">Notifications</span>
            </div>

              <div class="select-menu-list js-navigation-container" role="menu">

                <div class="select-menu-item js-navigation-item selected" role="menuitem" tabindex="0">
                  <svg class="octicon octicon-check select-menu-item-icon" viewBox="0 0 12 16" version="1.1" width="12" height="16" aria-hidden="true"><path fill-rule="evenodd" d="M12 5l-8 8-4-4 1.5-1.5L4 10l6.5-6.5L12 5z"></path></svg>
                  <div class="select-menu-item-text">
                    <input type="radio" name="do" id="do_included" value="included" checked="checked">
                    <span class="select-menu-item-heading">Not watching</span>
                    <span class="description">Be notified when participating or @mentioned.</span>
                    <span class="js-select-button-text hidden-select-button-text">
                      <svg class="octicon octicon-eye" viewBox="0 0 16 16" version="1.1" width="16" height="16" aria-hidden="true"><path fill-rule="evenodd" d="M8.06 2C3 2 0 8 0 8s3 6 8.06 6C13 14 16 8 16 8s-3-6-7.94-6zM8 12c-2.2 0-4-1.78-4-4 0-2.2 1.8-4 4-4 2.22 0 4 1.8 4 4 0 2.22-1.78 4-4 4zm2-4c0 1.11-.89 2-2 2-1.11 0-2-.89-2-2 0-1.11.89-2 2-2 1.11 0 2 .89 2 2z"></path></svg>
                      Watch
                    </span>
                  </div>
                </div>

                <div class="select-menu-item js-navigation-item " role="menuitem" tabindex="0">
                  <svg class="octicon octicon-check select-menu-item-icon" viewBox="0 0 12 16" version="1.1" width="12" height="16" aria-hidden="true"><path fill-rule="evenodd" d="M12 5l-8 8-4-4 1.5-1.5L4 10l6.5-6.5L12 5z"></path></svg>
                  <div class="select-menu-item-text">
                    <input type="radio" name="do" id="do_subscribed" value="subscribed">
                    <span class="select-menu-item-heading">Watching</span>
                    <span class="description">Be notified of all conversations.</span>
                    <span class="js-select-button-text hidden-select-button-text">
                      <svg class="octicon octicon-eye" viewBox="0 0 16 16" version="1.1" width="16" height="16" aria-hidden="true"><path fill-rule="evenodd" d="M8.06 2C3 2 0 8 0 8s3 6 8.06 6C13 14 16 8 16 8s-3-6-7.94-6zM8 12c-2.2 0-4-1.78-4-4 0-2.2 1.8-4 4-4 2.22 0 4 1.8 4 4 0 2.22-1.78 4-4 4zm2-4c0 1.11-.89 2-2 2-1.11 0-2-.89-2-2 0-1.11.89-2 2-2 1.11 0 2 .89 2 2z"></path></svg>
                        Unwatch
                    </span>
                  </div>
                </div>

                <div class="select-menu-item js-navigation-item " role="menuitem" tabindex="0">
                  <svg class="octicon octicon-check select-menu-item-icon" viewBox="0 0 12 16" version="1.1" width="12" height="16" aria-hidden="true"><path fill-rule="evenodd" d="M12 5l-8 8-4-4 1.5-1.5L4 10l6.5-6.5L12 5z"></path></svg>
                  <div class="select-menu-item-text">
                    <input type="radio" name="do" id="do_ignore" value="ignore">
                    <span class="select-menu-item-heading">Ignoring</span>
                    <span class="description">Never be notified.</span>
                    <span class="js-select-button-text hidden-select-button-text">
                      <svg class="octicon octicon-mute" viewBox="0 0 16 16" version="1.1" width="16" height="16" aria-hidden="true"><path fill-rule="evenodd" d="M8 2.81v10.38c0 .67-.81 1-1.28.53L3 10H1c-.55 0-1-.45-1-1V7c0-.55.45-1 1-1h2l3.72-3.72C7.19 1.81 8 2.14 8 2.81zm7.53 3.22l-1.06-1.06-1.97 1.97-1.97-1.97-1.06 1.06L11.44 8 9.47 9.97l1.06 1.06 1.97-1.97 1.97 1.97 1.06-1.06L13.56 8l1.97-1.97z"></path></svg>
                        Stop ignoring
                    </span>
                  </div>
                </div>

              </div>

            </div>
          </div>
        </div>
</form>
  </li>

  <li>
    
  <div class="js-toggler-container js-social-container starring-container ">
    <!-- '"` --><!-- </textarea></xmp> --><form class="starred js-social-form" action="https://github.com/griffithlab/rnaseq_tutorial/unstar" accept-charset="UTF-8" method="post"><input name="utf8" type="hidden" value="✓"><input type="hidden" name="authenticity_token" value="C5ns8QAkd83Fnk17V/werNYqyJAYpIrL8Ux0RvXU5Zc9ZVWxbC9n7GntXSzSqSxWOVEj8oab1ikhMUX1BLG84A==">
      <input type="hidden" name="context" value="repository">
      <button type="submit" class="btn btn-sm btn-with-count js-toggler-target" aria-label="Unstar this repository" title="Unstar griffithlab/rnaseq_tutorial" data-ga-click="Repository, click unstar button, action:wiki#show; text:Unstar">
        <svg class="octicon octicon-star" viewBox="0 0 14 16" version="1.1" width="14" height="16" aria-hidden="true"><path fill-rule="evenodd" d="M14 6l-4.9-.64L7 1 4.9 5.36 0 6l3.6 3.26L2.67 14 7 11.67 11.33 14l-.93-4.74L14 6z"></path></svg>
        Unstar
      </button>
        <a class="social-count js-social-count" href="https://github.com/griffithlab/rnaseq_tutorial/stargazers" aria-label="484 users starred this repository">
          484
        </a>
</form>
    <!-- '"` --><!-- </textarea></xmp> --><form class="unstarred js-social-form" action="https://github.com/griffithlab/rnaseq_tutorial/star" accept-charset="UTF-8" method="post"><input name="utf8" type="hidden" value="✓"><input type="hidden" name="authenticity_token" value="RtxsY1CuFRE8qJpvGpQY+nvdG3ihgyGstgy1wFka3MR/zotZIqV1SC8Wec53sxErBJ3JWtc4BXqVKz5NF9WGcA==">
      <input type="hidden" name="context" value="repository">
      <button type="submit" class="btn btn-sm btn-with-count js-toggler-target" aria-label="Star this repository" title="Star griffithlab/rnaseq_tutorial" data-ga-click="Repository, click star button, action:wiki#show; text:Star">
        <svg class="octicon octicon-star" viewBox="0 0 14 16" version="1.1" width="14" height="16" aria-hidden="true"><path fill-rule="evenodd" d="M14 6l-4.9-.64L7 1 4.9 5.36 0 6l3.6 3.26L2.67 14 7 11.67 11.33 14l-.93-4.74L14 6z"></path></svg>
        Star
      </button>
        <a class="social-count js-social-count" href="https://github.com/griffithlab/rnaseq_tutorial/stargazers" aria-label="484 users starred this repository">
          484
        </a>
</form>  </div>

  </li>

  <li>
          <!-- '"` --><!-- </textarea></xmp> --><form class="btn-with-count" action="https://github.com/griffithlab/rnaseq_tutorial/fork" accept-charset="UTF-8" method="post"><input name="utf8" type="hidden" value="✓"><input type="hidden" name="authenticity_token" value="+wQ9j3ZunMivR3nkNnu9UriQkMfxun+0dqZ6uyQ2RFdSUOU+c4zaxOFplrEEKIQX9VrfaIcBrJMglO4sW4S7aA==">
            <button type="submit" class="btn btn-sm btn-with-count" data-ga-click="Repository, show fork modal, action:wiki#show; text:Fork" title="Fork your own copy of griffithlab/rnaseq_tutorial to your account" aria-label="Fork your own copy of griffithlab/rnaseq_tutorial to your account">
              <svg class="octicon octicon-repo-forked" viewBox="0 0 10 16" version="1.1" width="10" height="16" aria-hidden="true"><path fill-rule="evenodd" d="M8 1a1.993 1.993 0 0 0-1 3.72V6L5 8 3 6V4.72A1.993 1.993 0 0 0 2 1a1.993 1.993 0 0 0-1 3.72V6.5l3 3v1.78A1.993 1.993 0 0 0 5 15a1.993 1.993 0 0 0 1-3.72V9.5l3-3V4.72A1.993 1.993 0 0 0 8 1zM2 4.2C1.34 4.2.8 3.65.8 3c0-.65.55-1.2 1.2-1.2.65 0 1.2.55 1.2 1.2 0 .65-.55 1.2-1.2 1.2zm3 10c-.66 0-1.2-.55-1.2-1.2 0-.65.55-1.2 1.2-1.2.65 0 1.2.55 1.2 1.2 0 .65-.55 1.2-1.2 1.2zm3-10c-.66 0-1.2-.55-1.2-1.2 0-.65.55-1.2 1.2-1.2.65 0 1.2.55 1.2 1.2 0 .65-.55 1.2-1.2 1.2z"></path></svg>
              Fork
            </button>
</form>
    <a href="https://github.com/griffithlab/rnaseq_tutorial/network" class="social-count" aria-label="293 users forked this repository">
      293
    </a>
  </li>
</ul>

      <h1 class="public ">
  <svg class="octicon octicon-repo" viewBox="0 0 12 16" version="1.1" width="12" height="16" aria-hidden="true"><path fill-rule="evenodd" d="M4 9H3V8h1v1zm0-3H3v1h1V6zm0-2H3v1h1V4zm0-2H3v1h1V2zm8-1v12c0 .55-.45 1-1 1H6v2l-1.5-1.5L3 16v-2H1c-.55 0-1-.45-1-1V1c0-.55.45-1 1-1h10c.55 0 1 .45 1 1zm-1 10H1v2h2v-1h3v1h5v-2zm0-10H2v9h9V1z"></path></svg>
  <span class="author" itemprop="author"><a class="url fn" rel="author" href="https://github.com/griffithlab">griffithlab</a></span><!--
--><span class="path-divider">/</span><!--
--><strong itemprop="name"><a data-pjax="#js-repo-pjax-container" href="https://github.com/griffithlab/rnaseq_tutorial">rnaseq_tutorial</a></strong>

</h1>

    </div>
    
<nav class="reponav js-repo-nav js-sidenav-container-pjax container" itemscope="" itemtype="http://schema.org/BreadcrumbList" role="navigation" data-pjax="#js-repo-pjax-container">

  <span itemscope="" itemtype="http://schema.org/ListItem" itemprop="itemListElement">
    <a class="js-selected-navigation-item reponav-item" itemprop="url" data-hotkey="g c" data-selected-links="repo_source repo_downloads repo_commits repo_releases repo_tags repo_branches repo_packages /griffithlab/rnaseq_tutorial" href="https://github.com/griffithlab/rnaseq_tutorial">
      <svg class="octicon octicon-code" viewBox="0 0 14 16" version="1.1" width="14" height="16" aria-hidden="true"><path fill-rule="evenodd" d="M9.5 3L8 4.5 11.5 8 8 11.5 9.5 13 14 8 9.5 3zm-5 0L0 8l4.5 5L6 11.5 2.5 8 6 4.5 4.5 3z"></path></svg>
      <span itemprop="name">Code</span>
      <meta itemprop="position" content="1">
</a>  </span>

    <span itemscope="" itemtype="http://schema.org/ListItem" itemprop="itemListElement">
      <a itemprop="url" data-hotkey="g i" class="js-selected-navigation-item reponav-item" data-selected-links="repo_issues repo_labels repo_milestones /griffithlab/rnaseq_tutorial/issues" href="https://github.com/griffithlab/rnaseq_tutorial/issues">
        <svg class="octicon octicon-issue-opened" viewBox="0 0 14 16" version="1.1" width="14" height="16" aria-hidden="true"><path fill-rule="evenodd" d="M7 2.3c3.14 0 5.7 2.56 5.7 5.7s-2.56 5.7-5.7 5.7A5.71 5.71 0 0 1 1.3 8c0-3.14 2.56-5.7 5.7-5.7zM7 1C3.14 1 0 4.14 0 8s3.14 7 7 7 7-3.14 7-7-3.14-7-7-7zm1 3H6v5h2V4zm0 6H6v2h2v-2z"></path></svg>
        <span itemprop="name">Issues</span>
        <span class="Counter">8</span>
        <meta itemprop="position" content="2">
</a>    </span>

  <span itemscope="" itemtype="http://schema.org/ListItem" itemprop="itemListElement">
    <a data-hotkey="g p" itemprop="url" class="js-selected-navigation-item reponav-item" data-selected-links="repo_pulls checks /griffithlab/rnaseq_tutorial/pulls" href="https://github.com/griffithlab/rnaseq_tutorial/pulls">
      <svg class="octicon octicon-git-pull-request" viewBox="0 0 12 16" version="1.1" width="12" height="16" aria-hidden="true"><path fill-rule="evenodd" d="M11 11.28V5c-.03-.78-.34-1.47-.94-2.06C9.46 2.35 8.78 2.03 8 2H7V0L4 3l3 3V4h1c.27.02.48.11.69.31.21.2.3.42.31.69v6.28A1.993 1.993 0 0 0 10 15a1.993 1.993 0 0 0 1-3.72zm-1 2.92c-.66 0-1.2-.55-1.2-1.2 0-.65.55-1.2 1.2-1.2.65 0 1.2.55 1.2 1.2 0 .65-.55 1.2-1.2 1.2zM4 3c0-1.11-.89-2-2-2a1.993 1.993 0 0 0-1 3.72v6.56A1.993 1.993 0 0 0 2 15a1.993 1.993 0 0 0 1-3.72V4.72c.59-.34 1-.98 1-1.72zm-.8 10c0 .66-.55 1.2-1.2 1.2-.65 0-1.2-.55-1.2-1.2 0-.65.55-1.2 1.2-1.2.65 0 1.2.55 1.2 1.2zM2 4.2C1.34 4.2.8 3.65.8 3c0-.65.55-1.2 1.2-1.2.65 0 1.2.55 1.2 1.2 0 .65-.55 1.2-1.2 1.2z"></path></svg>
      <span itemprop="name">Pull requests</span>
      <span class="Counter">0</span>
      <meta itemprop="position" content="3">
</a>  </span>

    <a data-hotkey="g b" class="js-selected-navigation-item reponav-item" data-selected-links="repo_projects new_repo_project repo_project /griffithlab/rnaseq_tutorial/projects" href="https://github.com/griffithlab/rnaseq_tutorial/projects">
      <svg class="octicon octicon-project" viewBox="0 0 15 16" version="1.1" width="15" height="16" aria-hidden="true"><path fill-rule="evenodd" d="M10 12h3V2h-3v10zm-4-2h3V2H6v8zm-4 4h3V2H2v12zm-1 1h13V1H1v14zM14 0H1a1 1 0 0 0-1 1v14a1 1 0 0 0 1 1h13a1 1 0 0 0 1-1V1a1 1 0 0 0-1-1z"></path></svg>
      Projects
      <span class="Counter">0</span>
</a>
    <a class="js-selected-navigation-item selected reponav-item" data-hotkey="g w" data-selected-links="repo_wiki /griffithlab/rnaseq_tutorial/wiki" href="https://github.com/griffithlab/rnaseq_tutorial/wiki">
      <svg class="octicon octicon-book" viewBox="0 0 16 16" version="1.1" width="16" height="16" aria-hidden="true"><path fill-rule="evenodd" d="M3 5h4v1H3V5zm0 3h4V7H3v1zm0 2h4V9H3v1zm11-5h-4v1h4V5zm0 2h-4v1h4V7zm0 2h-4v1h4V9zm2-6v9c0 .55-.45 1-1 1H9.5l-1 1-1-1H2c-.55 0-1-.45-1-1V3c0-.55.45-1 1-1h5.5l1 1 1-1H15c.55 0 1 .45 1 1zm-8 .5L7.5 3H2v9h6V3.5zm7-.5H9.5l-.5.5V12h6V3z"></path></svg>
      Wiki
</a>

  <a class="js-selected-navigation-item reponav-item" data-selected-links="repo_graphs repo_contributors dependency_graph pulse /griffithlab/rnaseq_tutorial/pulse" href="https://github.com/griffithlab/rnaseq_tutorial/pulse">
    <svg class="octicon octicon-graph" viewBox="0 0 16 16" version="1.1" width="16" height="16" aria-hidden="true"><path fill-rule="evenodd" d="M16 14v1H0V0h1v14h15zM5 13H3V8h2v5zm4 0H7V3h2v10zm4 0h-2V6h2v7z"></path></svg>
    Insights
</a>

</nav>


  </div>

<div class="container new-discussion-timeline experiment-repo-nav  ">
  <div class="repository-content ">

    
  <div id="wiki-wrapper" class="wiki-wrapper page">
  <div class="gh-header">
    <div class="gh-header-show">
      <h1 class="gh-header-title instapaper_title">IGV Tutorial</h1>
        <div class="gh-header-meta">
          Alex H. Wagner, PhD edited this page <relative-time datetime="2017-11-13T16:41:34Z" title="Nov 13, 2017, 10:41 AM CST">on Nov 13, 2017</relative-time>
          ·
          <a href="https://github.com/griffithlab/rnaseq_tutorial/wiki/IGV-Tutorial/_history" class="history">
            66 revisions
          </a>
        </div>
    </div>
  </div>
  <div id="wiki-content" class="wiki-content">
    <div class="has-rightbar">

    <div id="wiki-rightbar" class="wiki-rightbar">
      <div class="wiki-pages-box readability-sidebar boxed-group flush js-wiki-pages-box " role="navigation">
        
<h3 class="js-wiki-toggle-collapse wiki-auxiliary-content">
  <svg class="octicon octicon-triangle-down caret-expanded" viewBox="0 0 12 16" version="1.1" width="12" height="16" aria-hidden="true"><path fill-rule="evenodd" d="M0 5l6 6 6-6H0z"></path></svg>
  <svg class="octicon octicon-triangle-right caret-collapsed" viewBox="0 0 6 16" version="1.1" width="6" height="16" aria-hidden="true"><path fill-rule="evenodd" d="M0 14l6-6-6-6v12z"></path></svg>
  Pages <span class="Counter">44</span>
</h3>
<div class="boxed-group-inner wiki-auxiliary-content wiki-auxiliary-content-no-bg">
    <div class="filter-bar">
      <input type="text" id="wiki-pages-filter" class="form-control input-sm input-block js-filterable-field" placeholder="Find a Page…" aria-label="Find a Page…" tabindex="2">
    </div>

  <ul class="wiki-pages" data-filterable-for="wiki-pages-filter" data-filterable-type="substring">
    <li>
      <strong><a class="wiki-page-link" href="https://github.com/griffithlab/rnaseq_tutorial/wiki">Home</a></strong>
    </li>
    <li>
      <strong><a class="wiki-page-link" href="https://github.com/griffithlab/rnaseq_tutorial/wiki/Abbreviations">Abbreviations</a></strong>
    </li>
    <li>
      <strong><a class="wiki-page-link" href="https://github.com/griffithlab/rnaseq_tutorial/wiki/Adapter-Trim">Adapter Trim</a></strong>
    </li>
    <li>
      <strong><a class="wiki-page-link" href="https://github.com/griffithlab/rnaseq_tutorial/wiki/Alignment">Alignment</a></strong>
    </li>
    <li>
      <strong><a class="wiki-page-link" href="https://github.com/griffithlab/rnaseq_tutorial/wiki/Annotation">Annotation</a></strong>
    </li>
    <li>
      <strong><a class="wiki-page-link" href="https://github.com/griffithlab/rnaseq_tutorial/wiki/Authors">Authors</a></strong>
    </li>
    <li>
      <strong><a class="wiki-page-link" href="https://github.com/griffithlab/rnaseq_tutorial/wiki/AWS-Setup">AWS Setup</a></strong>
    </li>
    <li>
      <strong><a class="wiki-page-link" href="https://github.com/griffithlab/rnaseq_tutorial/wiki/Bioinformatics-Best-Practices">Bioinformatics Best Practices</a></strong>
    </li>
    <li>
      <strong><a class="wiki-page-link" href="https://github.com/griffithlab/rnaseq_tutorial/wiki/Citation">Citation</a></strong>
    </li>
    <li>
      <strong><a class="wiki-page-link" href="https://github.com/griffithlab/rnaseq_tutorial/wiki/de-novo-Transcript-Assembly">de novo Transcript Assembly</a></strong>
    </li>
    <li>
      <strong><a class="wiki-page-link" href="https://github.com/griffithlab/rnaseq_tutorial/wiki/DE-Visualization">DE Visualization</a></strong>
    </li>
    <li>
      <strong><a class="wiki-page-link" href="https://github.com/griffithlab/rnaseq_tutorial/wiki/Differential-Expression">Differential Expression</a></strong>
    </li>
    <li>
      <strong><a class="wiki-page-link" href="https://github.com/griffithlab/rnaseq_tutorial/wiki/Differential-Splicing">Differential Splicing</a></strong>
    </li>
    <li>
      <strong><a class="wiki-page-link" href="https://github.com/griffithlab/rnaseq_tutorial/wiki/Environment">Environment</a></strong>
    </li>
    <li>
      <strong><a class="wiki-page-link" href="https://github.com/griffithlab/rnaseq_tutorial/wiki/Expression">Expression</a></strong>
    </li>
    <li class="wiki-more-pages">
      <strong><a class="wiki-page-link" href="https://github.com/griffithlab/rnaseq_tutorial/wiki/IGV-Tutorial">IGV Tutorial</a></strong>
    </li>
    <li class="wiki-more-pages">
      <strong><a class="wiki-page-link" href="https://github.com/griffithlab/rnaseq_tutorial/wiki/Indexing">Indexing</a></strong>
    </li>
    <li class="wiki-more-pages">
      <strong><a class="wiki-page-link" href="https://github.com/griffithlab/rnaseq_tutorial/wiki/Installation">Installation</a></strong>
    </li>
    <li class="wiki-more-pages">
      <strong><a class="wiki-page-link" href="https://github.com/griffithlab/rnaseq_tutorial/wiki/Integrated-Assignment">Integrated Assignment</a></strong>
    </li>
    <li class="wiki-more-pages">
      <strong><a class="wiki-page-link" href="https://github.com/griffithlab/rnaseq_tutorial/wiki/Integrated-Assignment-Answers">Integrated Assignment Answers</a></strong>
    </li>
    <li class="wiki-more-pages">
      <strong><a class="wiki-page-link" href="https://github.com/griffithlab/rnaseq_tutorial/wiki/Intro-to-AWS-Cloud-Computing">Intro to AWS Cloud Computing</a></strong>
    </li>
    <li class="wiki-more-pages">
      <strong><a class="wiki-page-link" href="https://github.com/griffithlab/rnaseq_tutorial/wiki/Kallisto">Kallisto</a></strong>
    </li>
    <li class="wiki-more-pages">
      <strong><a class="wiki-page-link" href="https://github.com/griffithlab/rnaseq_tutorial/wiki/Lectures">Lectures</a></strong>
    </li>
    <li class="wiki-more-pages">
      <strong><a class="wiki-page-link" href="https://github.com/griffithlab/rnaseq_tutorial/wiki/Logging-into-Amazon-Cloud">Logging into Amazon Cloud</a></strong>
    </li>
    <li class="wiki-more-pages">
      <strong><a class="wiki-page-link" href="https://github.com/griffithlab/rnaseq_tutorial/wiki/PostAlignment-QC">PostAlignment QC</a></strong>
    </li>
    <li class="wiki-more-pages">
      <strong><a class="wiki-page-link" href="https://github.com/griffithlab/rnaseq_tutorial/wiki/PostAlignment-Visualization">PostAlignment Visualization</a></strong>
    </li>
    <li class="wiki-more-pages">
      <strong><a class="wiki-page-link" href="https://github.com/griffithlab/rnaseq_tutorial/wiki/PreAlignment-QC">PreAlignment QC</a></strong>
    </li>
    <li class="wiki-more-pages">
      <strong><a class="wiki-page-link" href="https://github.com/griffithlab/rnaseq_tutorial/wiki/Prerequisites">Prerequisites</a></strong>
    </li>
    <li class="wiki-more-pages">
      <strong><a class="wiki-page-link" href="https://github.com/griffithlab/rnaseq_tutorial/wiki/Proposed-Improvements">Proposed Improvements</a></strong>
    </li>
    <li class="wiki-more-pages">
      <strong><a class="wiki-page-link" href="https://github.com/griffithlab/rnaseq_tutorial/wiki/README">README</a></strong>
    </li>
    <li class="wiki-more-pages">
      <strong><a class="wiki-page-link" href="https://github.com/griffithlab/rnaseq_tutorial/wiki/Reference-Genome">Reference Genome</a></strong>
    </li>
    <li class="wiki-more-pages">
      <strong><a class="wiki-page-link" href="https://github.com/griffithlab/rnaseq_tutorial/wiki/Reference-Guided-Transcript-Assembly">Reference Guided Transcript Assembly</a></strong>
    </li>
    <li class="wiki-more-pages">
      <strong><a class="wiki-page-link" href="https://github.com/griffithlab/rnaseq_tutorial/wiki/Resources">Resources</a></strong>
    </li>
    <li class="wiki-more-pages">
      <strong><a class="wiki-page-link" href="https://github.com/griffithlab/rnaseq_tutorial/wiki/RNAseq-Data">RNAseq Data</a></strong>
    </li>
    <li class="wiki-more-pages">
      <strong><a class="wiki-page-link" href="https://github.com/griffithlab/rnaseq_tutorial/wiki/RSeQC-Tutorial-%5Boptional%5D">RSeQC Tutorial [optional]</a></strong>
    </li>
    <li class="wiki-more-pages">
      <strong><a class="wiki-page-link" href="https://github.com/griffithlab/rnaseq_tutorial/wiki/Saving-Your-Results">Saving Your Results</a></strong>
    </li>
    <li class="wiki-more-pages">
      <strong><a class="wiki-page-link" href="https://github.com/griffithlab/rnaseq_tutorial/wiki/Solutions">Solutions</a></strong>
    </li>
    <li class="wiki-more-pages">
      <strong><a class="wiki-page-link" href="https://github.com/griffithlab/rnaseq_tutorial/wiki/Syntax">Syntax</a></strong>
    </li>
    <li class="wiki-more-pages">
      <strong><a class="wiki-page-link" href="https://github.com/griffithlab/rnaseq_tutorial/wiki/Transcript-Assembly-Merge">Transcript Assembly Merge</a></strong>
    </li>
    <li class="wiki-more-pages">
      <strong><a class="wiki-page-link" href="https://github.com/griffithlab/rnaseq_tutorial/wiki/Transcript-Assembly-Visualization">Transcript Assembly Visualization</a></strong>
    </li>
    <li class="wiki-more-pages">
      <strong><a class="wiki-page-link" href="https://github.com/griffithlab/rnaseq_tutorial/wiki/Trinity-Assembly-And-Analysis">Trinity Assembly And Analysis</a></strong>
    </li>
    <li class="wiki-more-pages">
      <strong><a class="wiki-page-link" href="https://github.com/griffithlab/rnaseq_tutorial/wiki/Trinotate-Functional-Annotation">Trinotate Functional Annotation</a></strong>
    </li>
    <li class="wiki-more-pages">
      <strong><a class="wiki-page-link" href="https://github.com/griffithlab/rnaseq_tutorial/wiki/Unix-Bootcamp">Unix Bootcamp</a></strong>
    </li>
    <li class="wiki-more-pages">
      <strong><a class="wiki-page-link" href="https://github.com/griffithlab/rnaseq_tutorial/wiki/ZIA-Answers">ZIA Answers</a></strong>
    </li>
    <li class="wiki-more-pages-link">
      <a href="https://github.com/griffithlab/rnaseq_tutorial/wiki/IGV-Tutorial#" class="js-wiki-more-pages-link">
        Show 29 more pages…
      </a>
    </li>
  </ul>
</div>

      </div>


      <h5 class="mt-0 mb-2">Clone this wiki locally</h5>
      <div class="width-full input-group">
        <input id="wiki-clone-url" type="text" class="form-control input-sm text-small text-gray input-monospace js-url-field" aria-label="Clone URL for this wiki" value="https://github.com/griffithlab/rnaseq_tutorial.wiki.git" readonly="readonly">
        <span class="input-group-button">
          <clipboard-copy for="wiki-clone-url" aria-label="Copy to clipboard" class="btn btn-sm zeroclipboard-button" tabindex="0" role="button">
            <svg class="octicon octicon-clippy" viewBox="0 0 14 16" version="1.1" width="14" height="16" aria-hidden="true"><path fill-rule="evenodd" d="M2 13h4v1H2v-1zm5-6H2v1h5V7zm2 3V8l-3 3 3 3v-2h5v-2H9zM4.5 9H2v1h2.5V9zM2 12h2.5v-1H2v1zm9 1h1v2c-.02.28-.11.52-.3.7-.19.18-.42.28-.7.3H1c-.55 0-1-.45-1-1V4c0-.55.45-1 1-1h3c0-1.11.89-2 2-2 1.11 0 2 .89 2 2h3c.55 0 1 .45 1 1v5h-1V6H1v9h10v-2zM2 5h8c0-.55-.45-1-1-1H8c-.55 0-1-.45-1-1s-.45-1-1-1-1 .45-1 1-.45 1-1 1H3c-.55 0-1 .45-1 1z"></path></svg>
          </clipboard-copy>
        </span>
      </div>

        <a href="https://desktop.github.com/" class="btn btn-sm sidebar-button" title="Save this wiki to your computer and use it in GitHub Desktop." aria-label="Save this wiki to your computer and use it in GitHub Desktop.">
          <svg class="octicon octicon-desktop-download" viewBox="0 0 16 16" version="1.1" width="16" height="16" aria-hidden="true"><path fill-rule="evenodd" d="M4 6h3V0h2v6h3l-4 4-4-4zm11-4h-4v1h4v8H1V3h4V2H1c-.55 0-1 .45-1 1v9c0 .55.45 1 1 1h5.34c-.25.61-.86 1.39-2.34 2h8c-1.48-.61-2.09-1.39-2.34-2H15c.55 0 1-.45 1-1V3c0-.55-.45-1-1-1z"></path></svg>
          Clone in Desktop
        </a>

    </div>

    <div id="wiki-body" class="wiki-body gollum-markdown-content instapaper_body">
        <div class="markdown-body">
          <table>
<thead>
<tr>
<th align="center"><a class="internal present" href="https://github.com/griffithlab/rnaseq_tutorial/wiki/Alignment">Previous Section</a></th>
<th align="center"><a class="internal present" href="https://github.com/griffithlab/rnaseq_tutorial/wiki/IGV-Tutorial">This Section</a></th>
<th align="center"><a class="internal present" href="https://github.com/griffithlab/rnaseq_tutorial/wiki/PostAlignment-Visualization">Next Section</a></th>
</tr>
</thead>
<tbody>
<tr>
<td align="center"><a class="internal present" href="https://github.com/griffithlab/rnaseq_tutorial/wiki/Alignment">Alignment</a></td>
<td align="center"><a class="internal present" href="https://github.com/griffithlab/rnaseq_tutorial/wiki/IGV-Tutorial">IGV</a></td>
<td align="center"><a class="internal present" href="https://github.com/griffithlab/rnaseq_tutorial/wiki/PostAlignment-Visualization">Alignment Visualization</a></td>
</tr>
</tbody>
</table>
<hr>
<h2>
<a id="user-content-table-of-contents" class="anchor" href="https://github.com/griffithlab/rnaseq_tutorial/wiki/IGV-Tutorial#table-of-contents" aria-hidden="true"><svg class="octicon octicon-link" viewBox="0 0 16 16" version="1.1" width="16" height="16" aria-hidden="true"><path fill-rule="evenodd" d="M4 9h1v1H4c-1.5 0-3-1.69-3-3.5S2.55 3 4 3h4c1.45 0 3 1.69 3 3.5 0 1.41-.91 2.72-2 3.25V8.59c.58-.45 1-1.27 1-2.09C10 5.22 8.98 4 8 4H4c-.98 0-2 1.22-2 2.5S3 9 4 9zm9-3h-1v1h1c1 0 2 1.22 2 2.5S13.98 12 13 12H9c-.98 0-2-1.22-2-2.5 0-.83.42-1.64 1-2.09V6.25c-1.09.53-2 1.84-2 3.25C6 11.31 7.55 13 9 13h4c1.45 0 3-1.69 3-3.5S14.5 6 13 6z"></path></svg></a>Table of contents</h2>
<ol>
<li><a href="https://github.com/griffithlab/rnaseq_tutorial/wiki/IGV-Tutorial#introduction">Introduction</a></li>
<li><a href="https://github.com/griffithlab/rnaseq_tutorial/wiki/IGV-Tutorial#description-of-the-lab">Description of the lab</a></li>
<li><a href="https://github.com/griffithlab/rnaseq_tutorial/wiki/IGV-Tutorial#requirements">Requirements</a></li>
<li><a href="https://github.com/griffithlab/rnaseq_tutorial/wiki/IGV-Tutorial#compatibility">Compatibility</a></li>
<li><a href="https://github.com/griffithlab/rnaseq_tutorial/wiki/IGV-Tutorial#compatibility">Data Set for IGV</a></li>
<li><a href="https://github.com/griffithlab/rnaseq_tutorial/wiki/IGV-Tutorial#visualization-part-1-getting-familiar-with-igv">Visualization Part 1: Getting familiar with IGV</a></li>
<li><a href="https://github.com/griffithlab/rnaseq_tutorial/wiki/IGV-Tutorial#get-familiar-with-the-interface">Get familiar with the interface</a></li>
<li><a href="https://github.com/griffithlab/rnaseq_tutorial/wiki/IGV-Tutorial#load-a-genome-and-some-data-tracks">Load a Genome and Data Tracks</a></li>
<li><a href="https://github.com/griffithlab/rnaseq_tutorial/wiki/IGV-Tutorial#navigation">Navigation</a></li>
<li><a href="https://github.com/griffithlab/rnaseq_tutorial/wiki/IGV-Tutorial#region-lists">Region Lists</a></li>
<li><a href="https://github.com/griffithlab/rnaseq_tutorial/wiki/IGV-Tutorial#loading-read-alignments">Loading Read Alignments</a></li>
<li><a href="https://github.com/griffithlab/rnaseq_tutorial/wiki/IGV-Tutorial#visualizing-read-alignments">Visualizing read alignments</a></li>
<li><a href="https://github.com/griffithlab/rnaseq_tutorial/wiki/IGV-Tutorial#visualization-part-2-inspecting-snps-snvs-and-svs-">Visualization Part 2: Inspecting SNPs, SNVs, and SVs</a></li>
<li><a href="https://github.com/griffithlab/rnaseq_tutorial/wiki/IGV-Tutorial#neighbouring-somatic-snv-and-germline-snp">Neighbouring somatic SNV and germline SNP</a></li>
<li><a href="https://github.com/griffithlab/rnaseq_tutorial/wiki/IGV-Tutorial#homopolymer-repeat-with-indel">Homopolymer repeat with indel</a></li>
<li><a href="https://github.com/griffithlab/rnaseq_tutorial/wiki/IGV-Tutorial#coverage-by-gc">Coverage by GC</a></li>
<li><a href="https://github.com/griffithlab/rnaseq_tutorial/wiki/IGV-Tutorial#heterozygous-snps-on-different-alleles">Heterozygous SNPs on different alleles</a></li>
<li><a href="https://github.com/griffithlab/rnaseq_tutorial/wiki/IGV-Tutorial#low-mapping-quality">Low mapping quality</a></li>
<li><a href="https://github.com/griffithlab/rnaseq_tutorial/wiki/IGV-Tutorial#homozygous-deletion">Homozygous deletion</a></li>
<li><a href="https://github.com/griffithlab/rnaseq_tutorial/wiki/IGV-Tutorial#mis-alignment">Mis-alignment</a></li>
<li><a href="https://github.com/griffithlab/rnaseq_tutorial/wiki/IGV-Tutorial#translocation">Translocation</a></li>
<li><a href="https://github.com/griffithlab/rnaseq_tutorial/wiki/IGV-Tutorial#visualization-part-3-automating-tasks-in-igv">Visualization Part 3: Automating Tasks in IGV</a></li>
</ol>
<h2>
<a id="user-content-introduction" class="anchor" href="https://github.com/griffithlab/rnaseq_tutorial/wiki/IGV-Tutorial#introduction" aria-hidden="true"><svg class="octicon octicon-link" viewBox="0 0 16 16" version="1.1" width="16" height="16" aria-hidden="true"><path fill-rule="evenodd" d="M4 9h1v1H4c-1.5 0-3-1.69-3-3.5S2.55 3 4 3h4c1.45 0 3 1.69 3 3.5 0 1.41-.91 2.72-2 3.25V8.59c.58-.45 1-1.27 1-2.09C10 5.22 8.98 4 8 4H4c-.98 0-2 1.22-2 2.5S3 9 4 9zm9-3h-1v1h1c1 0 2 1.22 2 2.5S13.98 12 13 12H9c-.98 0-2-1.22-2-2.5 0-.83.42-1.64 1-2.09V6.25c-1.09.53-2 1.84-2 3.25C6 11.31 7.55 13 9 13h4c1.45 0 3-1.69 3-3.5S14.5 6 13 6z"></path></svg></a>Introduction</h2>
<h3>
<a id="user-content-description-of-the-lab" class="anchor" href="https://github.com/griffithlab/rnaseq_tutorial/wiki/IGV-Tutorial#description-of-the-lab" aria-hidden="true"><svg class="octicon octicon-link" viewBox="0 0 16 16" version="1.1" width="16" height="16" aria-hidden="true"><path fill-rule="evenodd" d="M4 9h1v1H4c-1.5 0-3-1.69-3-3.5S2.55 3 4 3h4c1.45 0 3 1.69 3 3.5 0 1.41-.91 2.72-2 3.25V8.59c.58-.45 1-1.27 1-2.09C10 5.22 8.98 4 8 4H4c-.98 0-2 1.22-2 2.5S3 9 4 9zm9-3h-1v1h1c1 0 2 1.22 2 2.5S13.98 12 13 12H9c-.98 0-2-1.22-2-2.5 0-.83.42-1.64 1-2.09V6.25c-1.09.53-2 1.84-2 3.25C6 11.31 7.55 13 9 13h4c1.45 0 3-1.69 3-3.5S14.5 6 13 6z"></path></svg></a>Description of the lab</h3>
<p>Welcome to the lab for <strong>Genome Visualization</strong>! This lab will introduce you to the [Integrative Genomics Viewer]<strong>(<a href="http://www.broadinstitute.org/igv/home" rel="nofollow">http://www.broadinstitute.org/igv/home</a>)</strong>, one of the most popular visualization tools for High Throughput Sequencing (HTS) data.</p>
<p>Lecture files that accompany this tutorial:</p>
<ul>
<li><a href="https://github.com/griffithlab/rnaseq_tutorial/wiki/LectureFiles/cshl/2016/IGV_Tutorial_Brief.pdf">IGV Lecture - Brief</a></li>
<li><a href="https://github.com/griffithlab/rnaseq_tutorial/wiki/LectureFiles/cshl/2016/IGV_Tutorial_Long_BroadInstitute.pdf">IGV Lecture - Long, from Broad Institute</a></li>
</ul>
<p>After this lab, you will be able to:</p>
<ul>
<li>Visualize a variety of genomic data</li>
<li>Quickly navigate around the genome</li>
<li>Visualize read alignments</li>
<li>Validate SNP/SNV calls and structural re-arrangements by eye</li>
</ul>
<p>Things to know before you start:</p>
<ul>
<li>The lab may take between <strong>1-2 hours</strong>, depending on your familiarity with genome browsing. Do not worry if you do not complete the lab. It will remain available to review later.</li>
<li>There are a few thought-provoking <strong>Questions</strong> or <strong>Notes</strong> pertaining to sections of the lab. These are <strong>optional</strong>, and may take more time, but are meant to help you better understand the visualizations you are seeing. These questions will be denoted by boxes, as follows:
<strong>Question(s):</strong>
</li>
</ul>
<pre><code>Thought-provoking question goes here
</code></pre>
<h3>
<a id="user-content-requirements" class="anchor" href="https://github.com/griffithlab/rnaseq_tutorial/wiki/IGV-Tutorial#requirements" aria-hidden="true"><svg class="octicon octicon-link" viewBox="0 0 16 16" version="1.1" width="16" height="16" aria-hidden="true"><path fill-rule="evenodd" d="M4 9h1v1H4c-1.5 0-3-1.69-3-3.5S2.55 3 4 3h4c1.45 0 3 1.69 3 3.5 0 1.41-.91 2.72-2 3.25V8.59c.58-.45 1-1.27 1-2.09C10 5.22 8.98 4 8 4H4c-.98 0-2 1.22-2 2.5S3 9 4 9zm9-3h-1v1h1c1 0 2 1.22 2 2.5S13.98 12 13 12H9c-.98 0-2-1.22-2-2.5 0-.83.42-1.64 1-2.09V6.25c-1.09.53-2 1.84-2 3.25C6 11.31 7.55 13 9 13h4c1.45 0 3-1.69 3-3.5S14.5 6 13 6z"></path></svg></a>Requirements</h3>
<ul>
<li><a href="http://www.broadinstitute.org/igv/home" rel="nofollow">Integrative Genomics Viewer</a></li>
<li>Ability to run Java</li>
<li>Note that while most tutorials in this course are performed on the cloud, IGV will always be run on your <strong>local</strong> machine</li>
</ul>
<h3>
<a id="user-content-compatibility" class="anchor" href="https://github.com/griffithlab/rnaseq_tutorial/wiki/IGV-Tutorial#compatibility" aria-hidden="true"><svg class="octicon octicon-link" viewBox="0 0 16 16" version="1.1" width="16" height="16" aria-hidden="true"><path fill-rule="evenodd" d="M4 9h1v1H4c-1.5 0-3-1.69-3-3.5S2.55 3 4 3h4c1.45 0 3 1.69 3 3.5 0 1.41-.91 2.72-2 3.25V8.59c.58-.45 1-1.27 1-2.09C10 5.22 8.98 4 8 4H4c-.98 0-2 1.22-2 2.5S3 9 4 9zm9-3h-1v1h1c1 0 2 1.22 2 2.5S13.98 12 13 12H9c-.98 0-2-1.22-2-2.5 0-.83.42-1.64 1-2.09V6.25c-1.09.53-2 1.84-2 3.25C6 11.31 7.55 13 9 13h4c1.45 0 3-1.69 3-3.5S14.5 6 13 6z"></path></svg></a>Compatibility</h3>
<p>This tutorial was intended for <strong>IGV v2.3</strong>, which is available on the <a href="http://www.broadinstitute.org/software/igv/download" rel="nofollow">IGV Download</a> page. It is <em>strongly</em> recommended that you use this version.</p>
<h3>
<a id="user-content-data-set-for-igv" class="anchor" href="https://github.com/griffithlab/rnaseq_tutorial/wiki/IGV-Tutorial#data-set-for-igv" aria-hidden="true"><svg class="octicon octicon-link" viewBox="0 0 16 16" version="1.1" width="16" height="16" aria-hidden="true"><path fill-rule="evenodd" d="M4 9h1v1H4c-1.5 0-3-1.69-3-3.5S2.55 3 4 3h4c1.45 0 3 1.69 3 3.5 0 1.41-.91 2.72-2 3.25V8.59c.58-.45 1-1.27 1-2.09C10 5.22 8.98 4 8 4H4c-.98 0-2 1.22-2 2.5S3 9 4 9zm9-3h-1v1h1c1 0 2 1.22 2 2.5S13.98 12 13 12H9c-.98 0-2-1.22-2-2.5 0-.83.42-1.64 1-2.09V6.25c-1.09.53-2 1.84-2 3.25C6 11.31 7.55 13 9 13h4c1.45 0 3-1.69 3-3.5S14.5 6 13 6z"></path></svg></a>Data Set for IGV</h3>
<p>We will be using publicly available Illumina sequence data from the HCC1143 cell line. The HCC1143 cell line was generated from a 52 year old caucasian woman with breast cancer. Additional information on this cell line can be found here: <a href="http://www.atcc.org/products/all/CRL-2321.aspx" rel="nofollow">HCC1143</a> (tumor, TNM stage IIA, grade 3, primary ductal carcinoma) and <a href="http://www.atcc.org/products/all/CRL-2362.aspx" rel="nofollow">HCC1143/BL</a> (matched normal EBV transformed lymphoblast cell line).</p>
<ul>
<li>Sequence read alignments generated from a cell line HCC1143 that have been filtered to this region:</li>
<li>Chromosome 21: 19,000,000-20,000,000</li>
<li><a href="https://xfer.genome.wustl.edu/gxfer1/project/gms/testdata/bams/hcc1143/HCC1143.normal.21.19M-20M.bam" rel="nofollow">HCC1143.normal.21.19M-20M.bam</a></li>
<li><a href="https://xfer.genome.wustl.edu/gxfer1/project/gms/testdata/bams/hcc1143/HCC1143.normal.21.19M-20M.bam.bai" rel="nofollow">HCC1143.normal.21.19M-20M.bam.bai</a></li>
</ul>
<h1>
<a id="user-content-visualization-part-1-getting-familiar-with-igv" class="anchor" href="https://github.com/griffithlab/rnaseq_tutorial/wiki/IGV-Tutorial#visualization-part-1-getting-familiar-with-igv" aria-hidden="true"><svg class="octicon octicon-link" viewBox="0 0 16 16" version="1.1" width="16" height="16" aria-hidden="true"><path fill-rule="evenodd" d="M4 9h1v1H4c-1.5 0-3-1.69-3-3.5S2.55 3 4 3h4c1.45 0 3 1.69 3 3.5 0 1.41-.91 2.72-2 3.25V8.59c.58-.45 1-1.27 1-2.09C10 5.22 8.98 4 8 4H4c-.98 0-2 1.22-2 2.5S3 9 4 9zm9-3h-1v1h1c1 0 2 1.22 2 2.5S13.98 12 13 12H9c-.98 0-2-1.22-2-2.5 0-.83.42-1.64 1-2.09V6.25c-1.09.53-2 1.84-2 3.25C6 11.31 7.55 13 9 13h4c1.45 0 3-1.69 3-3.5S14.5 6 13 6z"></path></svg></a>Visualization Part 1: Getting familiar with IGV</h1>
<p>We will be visualizing read alignments using <a href="http://www.broadinstitute.org/igv/home" rel="nofollow">IGV</a>, a popular visualization tool for HTS data.</p>
<p>First, lets familiarize ourselves with it.</p>
<h2>
<a id="user-content-get-familiar-with-the-interface" class="anchor" href="https://github.com/griffithlab/rnaseq_tutorial/wiki/IGV-Tutorial#get-familiar-with-the-interface" aria-hidden="true"><svg class="octicon octicon-link" viewBox="0 0 16 16" version="1.1" width="16" height="16" aria-hidden="true"><path fill-rule="evenodd" d="M4 9h1v1H4c-1.5 0-3-1.69-3-3.5S2.55 3 4 3h4c1.45 0 3 1.69 3 3.5 0 1.41-.91 2.72-2 3.25V8.59c.58-.45 1-1.27 1-2.09C10 5.22 8.98 4 8 4H4c-.98 0-2 1.22-2 2.5S3 9 4 9zm9-3h-1v1h1c1 0 2 1.22 2 2.5S13.98 12 13 12H9c-.98 0-2-1.22-2-2.5 0-.83.42-1.64 1-2.09V6.25c-1.09.53-2 1.84-2 3.25C6 11.31 7.55 13 9 13h4c1.45 0 3-1.69 3-3.5S14.5 6 13 6z"></path></svg></a>Get familiar with the interface</h2>
<h3>
<a id="user-content-load-a-genome-and-some-data-tracks" class="anchor" href="https://github.com/griffithlab/rnaseq_tutorial/wiki/IGV-Tutorial#load-a-genome-and-some-data-tracks" aria-hidden="true"><svg class="octicon octicon-link" viewBox="0 0 16 16" version="1.1" width="16" height="16" aria-hidden="true"><path fill-rule="evenodd" d="M4 9h1v1H4c-1.5 0-3-1.69-3-3.5S2.55 3 4 3h4c1.45 0 3 1.69 3 3.5 0 1.41-.91 2.72-2 3.25V8.59c.58-.45 1-1.27 1-2.09C10 5.22 8.98 4 8 4H4c-.98 0-2 1.22-2 2.5S3 9 4 9zm9-3h-1v1h1c1 0 2 1.22 2 2.5S13.98 12 13 12H9c-.98 0-2-1.22-2-2.5 0-.83.42-1.64 1-2.09V6.25c-1.09.53-2 1.84-2 3.25C6 11.31 7.55 13 9 13h4c1.45 0 3-1.69 3-3.5S14.5 6 13 6z"></path></svg></a>Load a Genome and some Data Tracks</h3>
<p>By default, IGV loads Human hg19. If you work with another version of the human genome, or another organism altogether, you can change the genome by clicking the drop down menu in the upper-left. For this lab, we will be using Human hg19.</p>
<p>We will also load additional tracks from <strong>Server</strong> using (<code>File</code> -&gt; <code>Load from Server...</code>):</p>
<ul>
<li>Ensembl genes (or your favourite source of gene annotations)</li>
<li>GC Percentage</li>
<li>dbSNP 1.3.1 or 1.3.7</li>
</ul>
<p><strong>Load hg19 genome and additional data tracks</strong>
<img src="./IGV Tutorial · griffithlab_rnaseq_tutorial Wiki_files/load.data.tracks.png" alt="Load hg19 genome and additional data tracks"></p>
<h3>
<a id="user-content-navigation" class="anchor" href="https://github.com/griffithlab/rnaseq_tutorial/wiki/IGV-Tutorial#navigation" aria-hidden="true"><svg class="octicon octicon-link" viewBox="0 0 16 16" version="1.1" width="16" height="16" aria-hidden="true"><path fill-rule="evenodd" d="M4 9h1v1H4c-1.5 0-3-1.69-3-3.5S2.55 3 4 3h4c1.45 0 3 1.69 3 3.5 0 1.41-.91 2.72-2 3.25V8.59c.58-.45 1-1.27 1-2.09C10 5.22 8.98 4 8 4H4c-.98 0-2 1.22-2 2.5S3 9 4 9zm9-3h-1v1h1c1 0 2 1.22 2 2.5S13.98 12 13 12H9c-.98 0-2-1.22-2-2.5 0-.83.42-1.64 1-2.09V6.25c-1.09.53-2 1.84-2 3.25C6 11.31 7.55 13 9 13h4c1.45 0 3-1.69 3-3.5S14.5 6 13 6z"></path></svg></a>Navigation</h3>
<p>You should see listing of chromosomes in this reference genome. Choose <em><strong>1</strong></em>, for chromosome 1.</p>
<p><strong>Chromosome chooser</strong>
<img src="./IGV Tutorial · griffithlab_rnaseq_tutorial Wiki_files/chromosomes.png" alt="Chromosome chooser"></p>
<p>Navigate to <strong>chr1:10,000-11,000</strong> by entering this into the location field (in the top-left corner of the interface) and clicking <code>Go</code>. This shows a window of chromosome 1 that is 1,000 base pairs wide and beginning at position 10,000.</p>
<p><strong>Navigition using Location text field. Sequence displayed as thin coloured rectangles.</strong>
<img src="./IGV Tutorial · griffithlab_rnaseq_tutorial Wiki_files/1.png" alt="Navigition using Location text field. Sequence displayed as thin coloured rectangles."></p>
<p>IGV displays the sequence of letters in a genome as a sequence of colours (e.g. A = green, C = blue, etc.). This makes repetitive sequences, like the ones found at the start of this region, easy to identify. Zoom in a bit more using the <code>+</code> button to see the individual bases of the reference genome sequence.</p>
<p>You can navigate to a gene of interest by typing it in the same box the genomic coordinates are in and pressing Enter/Return. Try it for your favourite gene, or <em>BRCA1</em> if you can not decide.</p>
<p><strong>Gene model</strong><br>
<img src="./IGV Tutorial · griffithlab_rnaseq_tutorial Wiki_files/gene_model.png" alt="Gene model"></p>
<p>Genes are represented as lines and boxes. Lines represent intronic regions, and boxes represent exonic regions. The arrows indicate the direction/strand of transcription for the gene. When an exon box become narrower in height, this indicates a UTR.</p>
<p>When loaded, tracks are stacked on top of each other. You can identify which track is which by consulting the label to the left of each track.</p>
<h2>
<a id="user-content-region-lists" class="anchor" href="https://github.com/griffithlab/rnaseq_tutorial/wiki/IGV-Tutorial#region-lists" aria-hidden="true"><svg class="octicon octicon-link" viewBox="0 0 16 16" version="1.1" width="16" height="16" aria-hidden="true"><path fill-rule="evenodd" d="M4 9h1v1H4c-1.5 0-3-1.69-3-3.5S2.55 3 4 3h4c1.45 0 3 1.69 3 3.5 0 1.41-.91 2.72-2 3.25V8.59c.58-.45 1-1.27 1-2.09C10 5.22 8.98 4 8 4H4c-.98 0-2 1.22-2 2.5S3 9 4 9zm9-3h-1v1h1c1 0 2 1.22 2 2.5S13.98 12 13 12H9c-.98 0-2-1.22-2-2.5 0-.83.42-1.64 1-2.09V6.25c-1.09.53-2 1.84-2 3.25C6 11.31 7.55 13 9 13h4c1.45 0 3-1.69 3-3.5S14.5 6 13 6z"></path></svg></a>Region Lists</h2>
<p>Sometimes, it is really useful to save where you are, or to load regions of interest. For this purpose, there is a <strong>Region Navigator</strong> in IGV. To access it, click <code>Regions</code> &gt; <code>Region Navigator</code>. While you browse around the genome, you can save some bookmarks by pressing the <code>Add</code> button at any time.</p>
<p><strong>Bookmarks in IGV</strong><br>
<img src="./IGV Tutorial · griffithlab_rnaseq_tutorial Wiki_files/bookmarks.png" alt="Bookmarks in IGV"></p>
<h2>
<a id="user-content-loading-read-alignments" class="anchor" href="https://github.com/griffithlab/rnaseq_tutorial/wiki/IGV-Tutorial#loading-read-alignments" aria-hidden="true"><svg class="octicon octicon-link" viewBox="0 0 16 16" version="1.1" width="16" height="16" aria-hidden="true"><path fill-rule="evenodd" d="M4 9h1v1H4c-1.5 0-3-1.69-3-3.5S2.55 3 4 3h4c1.45 0 3 1.69 3 3.5 0 1.41-.91 2.72-2 3.25V8.59c.58-.45 1-1.27 1-2.09C10 5.22 8.98 4 8 4H4c-.98 0-2 1.22-2 2.5S3 9 4 9zm9-3h-1v1h1c1 0 2 1.22 2 2.5S13.98 12 13 12H9c-.98 0-2-1.22-2-2.5 0-.83.42-1.64 1-2.09V6.25c-1.09.53-2 1.84-2 3.25C6 11.31 7.55 13 9 13h4c1.45 0 3-1.69 3-3.5S14.5 6 13 6z"></path></svg></a>Loading Read Alignments</h2>
<p>We will be using the breast cancer cell line HCC1143 to visualize alignments. For speed, only a small portion of chr21 will be loaded (19M:20M).</p>
<p><strong>HCC1143 Alignments to hg19:</strong></p>
<ul>
<li><a href="https://xfer.genome.wustl.edu/gxfer1/project/gms/testdata/bams/hcc1143/HCC1143.normal.21.19M-20M.bam" rel="nofollow">HCC1143.normal.21.19M-20M.bam</a></li>
<li><a href="https://xfer.genome.wustl.edu/gxfer1/project/gms/testdata/bams/hcc1143/HCC1143.normal.21.19M-20M.bam.bai" rel="nofollow">HCC1143.normal.21.19M-20M.bam.bai</a></li>
</ul>
<p>Copy the files to your local drive, and in IGV choose <code>File</code> &gt; <code>Load from File...</code>, select the bam file, and click <code>OK</code>. Note that the bam and index files must be in the same directory for IGV to load these properly.</p>
<p><strong>Load BAM track from File</strong>
<img src="./IGV Tutorial · griffithlab_rnaseq_tutorial Wiki_files/load_bam.png" alt="Load BAM track from File"></p>
<h2>
<a id="user-content-visualizing-read-alignments" class="anchor" href="https://github.com/griffithlab/rnaseq_tutorial/wiki/IGV-Tutorial#visualizing-read-alignments" aria-hidden="true"><svg class="octicon octicon-link" viewBox="0 0 16 16" version="1.1" width="16" height="16" aria-hidden="true"><path fill-rule="evenodd" d="M4 9h1v1H4c-1.5 0-3-1.69-3-3.5S2.55 3 4 3h4c1.45 0 3 1.69 3 3.5 0 1.41-.91 2.72-2 3.25V8.59c.58-.45 1-1.27 1-2.09C10 5.22 8.98 4 8 4H4c-.98 0-2 1.22-2 2.5S3 9 4 9zm9-3h-1v1h1c1 0 2 1.22 2 2.5S13.98 12 13 12H9c-.98 0-2-1.22-2-2.5 0-.83.42-1.64 1-2.09V6.25c-1.09.53-2 1.84-2 3.25C6 11.31 7.55 13 9 13h4c1.45 0 3-1.69 3-3.5S14.5 6 13 6z"></path></svg></a>Visualizing read alignments</h2>
<p>Navigate to a narrow window on chromosome 21: <code>chr21:19,480,041-19,480,386</code>.</p>
<p>To start our exploration, right click on the track-name, and select the following options:</p>
<ul>
<li>Sort alignments by <code>start location</code>
</li>
<li>Group alignments by <code>pair orientation</code>
</li>
</ul>
<p>Experiment with the various settings by right clicking the read alignment track and toggling the options. Think about which would be best for specific tasks (e.g. quality control, SNP calling, CNV finding).</p>
<p><strong>Changing how read alignments are sorted, grouped, and colored</strong><br>
<img src="./IGV Tutorial · griffithlab_rnaseq_tutorial Wiki_files/sort_and_group.png" alt="Changing how read alignments are sorted, grouped, and colored"></p>
<p>You will see reads represented by grey or white bars stacked on top of each other, where they were aligned to the reference genome. The reads are pointed to indicate their orientation (i.e. the strand on which they are mapped). Mouse over any read and notice that a lot of information is available. To toggle read display from <code>hover</code> to <code>click</code>, select the yellow box and change the setting.</p>
<p><strong>Changing how read information is shown (i.e. on hover, click, never)</strong>
<img src="./IGV Tutorial · griffithlab_rnaseq_tutorial Wiki_files/show_details_on_click.png" alt="Changing how read information is shown (i.e. on hover, click, never)"></p>
<p>Once you select a read, you will learn what many of these metrics mean, and how to use them to assess the quality of your datasets.  At each base that the read sequence <strong>mismatches</strong> the reference, the colour of the base represents the letter that exists in the read (using the same colour legend used for displaying the reference).</p>
<p><strong>Viewing read information for a single aligned read</strong>
<img src="./IGV Tutorial · griffithlab_rnaseq_tutorial Wiki_files/click_read.png" alt="Viewing read information for a single aligned read"></p>
<h1>
<a id="user-content-visualization-part-2-inspecting-snps-snvs-and-svs" class="anchor" href="https://github.com/griffithlab/rnaseq_tutorial/wiki/IGV-Tutorial#visualization-part-2-inspecting-snps-snvs-and-svs" aria-hidden="true"><svg class="octicon octicon-link" viewBox="0 0 16 16" version="1.1" width="16" height="16" aria-hidden="true"><path fill-rule="evenodd" d="M4 9h1v1H4c-1.5 0-3-1.69-3-3.5S2.55 3 4 3h4c1.45 0 3 1.69 3 3.5 0 1.41-.91 2.72-2 3.25V8.59c.58-.45 1-1.27 1-2.09C10 5.22 8.98 4 8 4H4c-.98 0-2 1.22-2 2.5S3 9 4 9zm9-3h-1v1h1c1 0 2 1.22 2 2.5S13.98 12 13 12H9c-.98 0-2-1.22-2-2.5 0-.83.42-1.64 1-2.09V6.25c-1.09.53-2 1.84-2 3.25C6 11.31 7.55 13 9 13h4c1.45 0 3-1.69 3-3.5S14.5 6 13 6z"></path></svg></a>Visualization Part 2: Inspecting SNPs, SNVs, and SVs</h1>
<p>In this section we will be looking in detail at 8 positions in the genome, and determining whether they represent real events or artifacts.</p>
<h2>
<a id="user-content-two-neighbouring-snps" class="anchor" href="https://github.com/griffithlab/rnaseq_tutorial/wiki/IGV-Tutorial#two-neighbouring-snps" aria-hidden="true"><svg class="octicon octicon-link" viewBox="0 0 16 16" version="1.1" width="16" height="16" aria-hidden="true"><path fill-rule="evenodd" d="M4 9h1v1H4c-1.5 0-3-1.69-3-3.5S2.55 3 4 3h4c1.45 0 3 1.69 3 3.5 0 1.41-.91 2.72-2 3.25V8.59c.58-.45 1-1.27 1-2.09C10 5.22 8.98 4 8 4H4c-.98 0-2 1.22-2 2.5S3 9 4 9zm9-3h-1v1h1c1 0 2 1.22 2 2.5S13.98 12 13 12H9c-.98 0-2-1.22-2-2.5 0-.83.42-1.64 1-2.09V6.25c-1.09.53-2 1.84-2 3.25C6 11.31 7.55 13 9 13h4c1.45 0 3-1.69 3-3.5S14.5 6 13 6z"></path></svg></a>Two neighbouring SNPs</h2>
<ul>
<li>Navigate to region <code>chr21:19,479,237-19,479,814</code>
</li>
<li>Note two heterozygous variants, one corresponds to a known dbSNP (<code>G/T</code> on the right) the other does not (<code>C/T</code> on the left)</li>
<li>Zoom in and center on the <code>C/T</code> SNV on the left, sort by base (window <code>chr21:19,479,321</code> is the SNV position)</li>
<li>Sort alignments by <code>base</code>
</li>
<li>Color alignments by <code>read strand</code>
</li>
</ul>
<p><strong>Example1. Good quality SNVs/SNPs</strong>
<img src="./IGV Tutorial · griffithlab_rnaseq_tutorial Wiki_files/example1_color.png" alt="Example1. Good quality SNVs/SNPs"></p>
<p><strong>Notes:</strong></p>
<ul>
<li>High base qualities in all reads except one (where the alt allele is the last base of the read)</li>
<li>Good mapping quality of reads, no strand bias, allele frequency consistent with heterozygous mutation</li>
</ul>
<p><strong>Question(s):</strong></p>
<pre><code>* What does *Shade base by quality* do? How might this be helpful?
* How does Color by *read strand* help?
</code></pre>
<h2>
<a id="user-content-homopolymer-region-with-indel" class="anchor" href="https://github.com/griffithlab/rnaseq_tutorial/wiki/IGV-Tutorial#homopolymer-region-with-indel" aria-hidden="true"><svg class="octicon octicon-link" viewBox="0 0 16 16" version="1.1" width="16" height="16" aria-hidden="true"><path fill-rule="evenodd" d="M4 9h1v1H4c-1.5 0-3-1.69-3-3.5S2.55 3 4 3h4c1.45 0 3 1.69 3 3.5 0 1.41-.91 2.72-2 3.25V8.59c.58-.45 1-1.27 1-2.09C10 5.22 8.98 4 8 4H4c-.98 0-2 1.22-2 2.5S3 9 4 9zm9-3h-1v1h1c1 0 2 1.22 2 2.5S13.98 12 13 12H9c-.98 0-2-1.22-2-2.5 0-.83.42-1.64 1-2.09V6.25c-1.09.53-2 1.84-2 3.25C6 11.31 7.55 13 9 13h4c1.45 0 3-1.69 3-3.5S14.5 6 13 6z"></path></svg></a>Homopolymer region with indel</h2>
<p>Navigate to position <code>chr21:19,518,412-19,518,497</code></p>
<p><strong>Example 2a</strong></p>
<ul>
<li>Group alignments by <code>read strand</code>
</li>
<li>Center on the <code>A</code> within the homopolymer run (<code>chr21:19,518,470</code>), and <code>Sort alignments by</code> -&gt; <code>base</code>
</li>
</ul>
<p><img src="./IGV Tutorial · griffithlab_rnaseq_tutorial Wiki_files/example2a.png" alt="Example 2a"></p>
<p><strong>Example 2b</strong></p>
<ul>
<li>Center on the one base deletion (<code>chr21:19,518,452</code>), and <code>Sort alignments by</code> -&gt; <code>base</code>
</li>
</ul>
<p><img src="./IGV Tutorial · griffithlab_rnaseq_tutorial Wiki_files/example2b.png" alt="Example 2b"></p>
<p><strong>Notes:</strong></p>
<ul>
<li>The alt allele is either a deletion or insertion of one or two <code>T</code>s</li>
<li>The remaining bases are mismatched, because the alignment is now out of sync</li>
<li>The dpSNP entry at this location (rs74604068) is an A-&gt;T, and in all likelihood an artifact</li>
<li>i.e. the common variants from dbSNP include some cases that are actually common misalignments caused by repeats</li>
</ul>
<h2>
<a id="user-content-coverage-by-gc" class="anchor" href="https://github.com/griffithlab/rnaseq_tutorial/wiki/IGV-Tutorial#coverage-by-gc" aria-hidden="true"><svg class="octicon octicon-link" viewBox="0 0 16 16" version="1.1" width="16" height="16" aria-hidden="true"><path fill-rule="evenodd" d="M4 9h1v1H4c-1.5 0-3-1.69-3-3.5S2.55 3 4 3h4c1.45 0 3 1.69 3 3.5 0 1.41-.91 2.72-2 3.25V8.59c.58-.45 1-1.27 1-2.09C10 5.22 8.98 4 8 4H4c-.98 0-2 1.22-2 2.5S3 9 4 9zm9-3h-1v1h1c1 0 2 1.22 2 2.5S13.98 12 13 12H9c-.98 0-2-1.22-2-2.5 0-.83.42-1.64 1-2.09V6.25c-1.09.53-2 1.84-2 3.25C6 11.31 7.55 13 9 13h4c1.45 0 3-1.69 3-3.5S14.5 6 13 6z"></path></svg></a>Coverage by GC</h2>
<p>Navigate to position <code>chr21:19,611,925-19,631,555</code>. Note that the range contains areas where coverage drops to zero in a few places.</p>
<p><strong>Example 3</strong></p>
<ul>
<li>Use <code>Collapsed</code> view</li>
<li>Use <code>Color alignments by</code> -&gt; <code>insert size and pair orientation</code>
</li>
<li>Load GC track</li>
<li>See concordance of coverage with GC content</li>
</ul>
<p><img src="./IGV Tutorial · griffithlab_rnaseq_tutorial Wiki_files/example3.png" alt="Example 3"></p>
<p><strong>Question:</strong></p>
<pre><code>* Why are there blue and red reads throughout the alignments?
</code></pre>
<h2>
<a id="user-content-heterozygous-snps-on-different-alleles" class="anchor" href="https://github.com/griffithlab/rnaseq_tutorial/wiki/IGV-Tutorial#heterozygous-snps-on-different-alleles" aria-hidden="true"><svg class="octicon octicon-link" viewBox="0 0 16 16" version="1.1" width="16" height="16" aria-hidden="true"><path fill-rule="evenodd" d="M4 9h1v1H4c-1.5 0-3-1.69-3-3.5S2.55 3 4 3h4c1.45 0 3 1.69 3 3.5 0 1.41-.91 2.72-2 3.25V8.59c.58-.45 1-1.27 1-2.09C10 5.22 8.98 4 8 4H4c-.98 0-2 1.22-2 2.5S3 9 4 9zm9-3h-1v1h1c1 0 2 1.22 2 2.5S13.98 12 13 12H9c-.98 0-2-1.22-2-2.5 0-.83.42-1.64 1-2.09V6.25c-1.09.53-2 1.84-2 3.25C6 11.31 7.55 13 9 13h4c1.45 0 3-1.69 3-3.5S14.5 6 13 6z"></path></svg></a>Heterozygous SNPs on different alleles</h2>
<p>Navigate to region <code>chr21:19,666,833-19,667,007</code></p>
<p><strong>Example 4</strong></p>
<ul>
<li>Sort by base (at position <code>chr21:19,666,901</code>)</li>
</ul>
<p><img src="./IGV Tutorial · griffithlab_rnaseq_tutorial Wiki_files/example4.png" alt="Example 4"></p>
<p><strong>Note:</strong></p>
<ul>
<li>There is no linkage between alleles for these two SNPs because reads covering both only contain one or the other</li>
</ul>
<h2>
<a id="user-content-low-mapping-quality" class="anchor" href="https://github.com/griffithlab/rnaseq_tutorial/wiki/IGV-Tutorial#low-mapping-quality" aria-hidden="true"><svg class="octicon octicon-link" viewBox="0 0 16 16" version="1.1" width="16" height="16" aria-hidden="true"><path fill-rule="evenodd" d="M4 9h1v1H4c-1.5 0-3-1.69-3-3.5S2.55 3 4 3h4c1.45 0 3 1.69 3 3.5 0 1.41-.91 2.72-2 3.25V8.59c.58-.45 1-1.27 1-2.09C10 5.22 8.98 4 8 4H4c-.98 0-2 1.22-2 2.5S3 9 4 9zm9-3h-1v1h1c1 0 2 1.22 2 2.5S13.98 12 13 12H9c-.98 0-2-1.22-2-2.5 0-.83.42-1.64 1-2.09V6.25c-1.09.53-2 1.84-2 3.25C6 11.31 7.55 13 9 13h4c1.45 0 3-1.69 3-3.5S14.5 6 13 6z"></path></svg></a>Low mapping quality</h2>
<p>Navigate to region <code>chr21:19,800,320-19,818,162</code></p>
<ul>
<li>Load repeat track (<code>File</code> -&gt; <code>Load from server...</code>)</li>
</ul>
<p><strong>Load repeats</strong><br>
<img src="./IGV Tutorial · griffithlab_rnaseq_tutorial Wiki_files/load_repeats.png" alt="Load repeats"></p>
<p><strong>Example 5</strong>
<img src="./IGV Tutorial · griffithlab_rnaseq_tutorial Wiki_files/example5.png" alt="Example 5"></p>
<p><strong>Notes:</strong></p>
<ul>
<li>Mapping quality plunges in all reads (white instead of grey).  Once we load repeat elements, we see that there are two LINE elements that cause this.</li>
</ul>
<h2>
<a id="user-content-homozygous-deletion" class="anchor" href="https://github.com/griffithlab/rnaseq_tutorial/wiki/IGV-Tutorial#homozygous-deletion" aria-hidden="true"><svg class="octicon octicon-link" viewBox="0 0 16 16" version="1.1" width="16" height="16" aria-hidden="true"><path fill-rule="evenodd" d="M4 9h1v1H4c-1.5 0-3-1.69-3-3.5S2.55 3 4 3h4c1.45 0 3 1.69 3 3.5 0 1.41-.91 2.72-2 3.25V8.59c.58-.45 1-1.27 1-2.09C10 5.22 8.98 4 8 4H4c-.98 0-2 1.22-2 2.5S3 9 4 9zm9-3h-1v1h1c1 0 2 1.22 2 2.5S13.98 12 13 12H9c-.98 0-2-1.22-2-2.5 0-.83.42-1.64 1-2.09V6.25c-1.09.53-2 1.84-2 3.25C6 11.31 7.55 13 9 13h4c1.45 0 3-1.69 3-3.5S14.5 6 13 6z"></path></svg></a>Homozygous deletion</h2>
<p>Navigate to region <code>chr21:19,324,469-19,331,468</code></p>
<p><strong>Example 6</strong></p>
<ul>
<li>Turn on <code>View as Pairs</code> and <code>Expanded</code> view</li>
<li>Use <code>Color alignments by</code> -&gt; <code>insert size and pair orientation</code>
</li>
<li>Sort reads by insert size</li>
<li>Click on a red read pair to pull up information on alignments</li>
</ul>
<p><img src="./IGV Tutorial · griffithlab_rnaseq_tutorial Wiki_files/example6.png" alt="Example 6"></p>
<p><strong>Notes:</strong></p>
<ul>
<li>Typical insert size of read pair in the vicinity: 350bp</li>
<li>Insert size of red read pairs: 2,875bp</li>
<li>This corresponds to a homozygous deletion of 2.5kb</li>
</ul>
<h2>
<a id="user-content-mis-alignment" class="anchor" href="https://github.com/griffithlab/rnaseq_tutorial/wiki/IGV-Tutorial#mis-alignment" aria-hidden="true"><svg class="octicon octicon-link" viewBox="0 0 16 16" version="1.1" width="16" height="16" aria-hidden="true"><path fill-rule="evenodd" d="M4 9h1v1H4c-1.5 0-3-1.69-3-3.5S2.55 3 4 3h4c1.45 0 3 1.69 3 3.5 0 1.41-.91 2.72-2 3.25V8.59c.58-.45 1-1.27 1-2.09C10 5.22 8.98 4 8 4H4c-.98 0-2 1.22-2 2.5S3 9 4 9zm9-3h-1v1h1c1 0 2 1.22 2 2.5S13.98 12 13 12H9c-.98 0-2-1.22-2-2.5 0-.83.42-1.64 1-2.09V6.25c-1.09.53-2 1.84-2 3.25C6 11.31 7.55 13 9 13h4c1.45 0 3-1.69 3-3.5S14.5 6 13 6z"></path></svg></a>Mis-alignment</h2>
<p>Navigate to region <code>chr21:19,102,154-19,103,108</code></p>
<p><strong>Example 7</strong>
<img src="./IGV Tutorial · griffithlab_rnaseq_tutorial Wiki_files/example7.png" alt="Example 7"></p>
<p><strong>Notes:</strong></p>
<ul>
<li>This is a position where AluY element causes mis-alignment.</li>
<li>Misaligned reads have mismatches to the reference and well-aligned reads have partners on other chromosomes where additional ALuY elements are encoded.</li>
<li>Zoom out until you can clearly see the contrast between the difficult alignment region (corresponding to an AluY) and regions with clean alignments on either side</li>
</ul>
<h2>
<a id="user-content-translocation" class="anchor" href="https://github.com/griffithlab/rnaseq_tutorial/wiki/IGV-Tutorial#translocation" aria-hidden="true"><svg class="octicon octicon-link" viewBox="0 0 16 16" version="1.1" width="16" height="16" aria-hidden="true"><path fill-rule="evenodd" d="M4 9h1v1H4c-1.5 0-3-1.69-3-3.5S2.55 3 4 3h4c1.45 0 3 1.69 3 3.5 0 1.41-.91 2.72-2 3.25V8.59c.58-.45 1-1.27 1-2.09C10 5.22 8.98 4 8 4H4c-.98 0-2 1.22-2 2.5S3 9 4 9zm9-3h-1v1h1c1 0 2 1.22 2 2.5S13.98 12 13 12H9c-.98 0-2-1.22-2-2.5 0-.83.42-1.64 1-2.09V6.25c-1.09.53-2 1.84-2 3.25C6 11.31 7.55 13 9 13h4c1.45 0 3-1.69 3-3.5S14.5 6 13 6z"></path></svg></a>Translocation</h2>
<p>Navigate to region <code>chr21:19,089,694-19,095,362</code></p>
<p><strong>Example 8</strong></p>
<ul>
<li>Expanded view</li>
<li>
<code>Group alignments by</code> -&gt; <code>pair orientation</code>
</li>
<li>
<code>Color alignments by</code> -&gt; <code>pair orientation</code>
</li>
</ul>
<p><img src="./IGV Tutorial · griffithlab_rnaseq_tutorial Wiki_files/example8.png" alt="Example 8"></p>
<p><strong>Notes:</strong></p>
<ul>
<li>Many reads with mismatches to reference</li>
<li>Read pairs in RL pattern (instead of LR pattern)</li>
<li>Region is flanked by reads with poor mapping quality (white instead of grey)</li>
<li>Presence of reads with pairs on other chromosomes (coloured reads at the bottom when scrolling down)</li>
</ul>
<h1>
<a id="user-content-visualization-part-3-automating-tasks-in-igv" class="anchor" href="https://github.com/griffithlab/rnaseq_tutorial/wiki/IGV-Tutorial#visualization-part-3-automating-tasks-in-igv" aria-hidden="true"><svg class="octicon octicon-link" viewBox="0 0 16 16" version="1.1" width="16" height="16" aria-hidden="true"><path fill-rule="evenodd" d="M4 9h1v1H4c-1.5 0-3-1.69-3-3.5S2.55 3 4 3h4c1.45 0 3 1.69 3 3.5 0 1.41-.91 2.72-2 3.25V8.59c.58-.45 1-1.27 1-2.09C10 5.22 8.98 4 8 4H4c-.98 0-2 1.22-2 2.5S3 9 4 9zm9-3h-1v1h1c1 0 2 1.22 2 2.5S13.98 12 13 12H9c-.98 0-2-1.22-2-2.5 0-.83.42-1.64 1-2.09V6.25c-1.09.53-2 1.84-2 3.25C6 11.31 7.55 13 9 13h4c1.45 0 3-1.69 3-3.5S14.5 6 13 6z"></path></svg></a>Visualization Part 3: Automating Tasks in IGV</h1>
<p>We can use the Tools menu to invoke running a batch script. Batch scripts are described on the IGV website:</p>
<ul>
<li>Batch file requirements: <a href="https://www.broadinstitute.org/igv/batch" rel="nofollow">https://www.broadinstitute.org/igv/batch</a>
</li>
<li>Commands recognized in a batch script: <a href="https://www.broadinstitute.org/software/igv/PortCommands" rel="nofollow">https://www.broadinstitute.org/software/igv/PortCommands</a>
</li>
<li>We also need to provide sample attribute file as described here: <a href="http://www.broadinstitute.org/software/igv/?q=SampleInformation" rel="nofollow">http://www.broadinstitute.org/software/igv/?q=SampleInformation</a>
</li>
</ul>
<p>Download the batch script and the attribute file for our dataset:</p>
<ul>
<li>Batch script: <a href="https://raw.githubusercontent.com/griffithlab/rnaseq_tutorial/master/scripts/Run_batch_IGV_snapshots.txt" rel="nofollow">Run_batch_IGV_snapshots.txt</a>
</li>
<li>Attribute file: <a href="https://raw.githubusercontent.com/griffithlab/rnaseq_tutorial/master/scripts/Igv_HCC1143_attributes.txt" rel="nofollow">Igv_HCC1143_attributes.txt</a>
</li>
</ul>
<p>Now run the file from the <code>Tools</code> menu:</p>
<p><strong>Automation</strong>
<img src="./IGV Tutorial · griffithlab_rnaseq_tutorial Wiki_files/run_batch_script.png" alt="Automation"></p>
<p><strong>Notes:</strong></p>
<ul>
<li>This script will navigate automatically to each location in the lab</li>
<li>A screenshot will be taken and saved to the screenshots directory specified</li>
</ul>
<h2>
<a id="user-content-contributorsacknowledgements" class="anchor" href="https://github.com/griffithlab/rnaseq_tutorial/wiki/IGV-Tutorial#contributorsacknowledgements" aria-hidden="true"><svg class="octicon octicon-link" viewBox="0 0 16 16" version="1.1" width="16" height="16" aria-hidden="true"><path fill-rule="evenodd" d="M4 9h1v1H4c-1.5 0-3-1.69-3-3.5S2.55 3 4 3h4c1.45 0 3 1.69 3 3.5 0 1.41-.91 2.72-2 3.25V8.59c.58-.45 1-1.27 1-2.09C10 5.22 8.98 4 8 4H4c-.98 0-2 1.22-2 2.5S3 9 4 9zm9-3h-1v1h1c1 0 2 1.22 2 2.5S13.98 12 13 12H9c-.98 0-2-1.22-2-2.5 0-.83.42-1.64 1-2.09V6.25c-1.09.53-2 1.84-2 3.25C6 11.31 7.55 13 9 13h4c1.45 0 3-1.69 3-3.5S14.5 6 13 6z"></path></svg></a>Contributors/acknowledgements</h2>
<p>Malachi Griffith, Sorana Morrissy, Jim Robinson, Ben Ainscough, Jason Walker, Obi Griffith</p>
<hr>
<table>
<thead>
<tr>
<th align="center"><a class="internal present" href="https://github.com/griffithlab/rnaseq_tutorial/wiki/Alignment">Previous Section</a></th>
<th align="center"><a class="internal present" href="https://github.com/griffithlab/rnaseq_tutorial/wiki/IGV-Tutorial">This Section</a></th>
<th align="center"><a class="internal present" href="https://github.com/griffithlab/rnaseq_tutorial/wiki/PostAlignment-Visualization">Next Section</a></th>
</tr>
</thead>
<tbody>
<tr>
<td align="center"><a class="internal present" href="https://github.com/griffithlab/rnaseq_tutorial/wiki/Alignment">Alignment</a></td>
<td align="center"><a class="internal present" href="https://github.com/griffithlab/rnaseq_tutorial/wiki/IGV-Tutorial">IGV</a></td>
<td align="center"><a class="internal present" href="https://github.com/griffithlab/rnaseq_tutorial/wiki/PostAlignment-Visualization">Alignment Visualization</a></td>
</tr>
</tbody>
</table>

        </div>

        <div id="wiki-footer" class="wiki-footer gollum-markdown-content boxed-group">
          <div class="boxed-group-inner wiki-auxiliary-content  markdown-body">
            <h3>
<a id="user-content-want-to-contribute-to-this-wiki" class="anchor" href="https://github.com/griffithlab/rnaseq_tutorial/wiki/IGV-Tutorial#want-to-contribute-to-this-wiki" aria-hidden="true"><svg class="octicon octicon-link" viewBox="0 0 16 16" version="1.1" width="16" height="16" aria-hidden="true"><path fill-rule="evenodd" d="M4 9h1v1H4c-1.5 0-3-1.69-3-3.5S2.55 3 4 3h4c1.45 0 3 1.69 3 3.5 0 1.41-.91 2.72-2 3.25V8.59c.58-.45 1-1.27 1-2.09C10 5.22 8.98 4 8 4H4c-.98 0-2 1.22-2 2.5S3 9 4 9zm9-3h-1v1h1c1 0 2 1.22 2 2.5S13.98 12 13 12H9c-.98 0-2-1.22-2-2.5 0-.83.42-1.64 1-2.09V6.25c-1.09.53-2 1.84-2 3.25C6 11.31 7.55 13 9 13h4c1.45 0 3-1.69 3-3.5S14.5 6 13 6z"></path></svg></a>Want to contribute to this Wiki?</h3>
<p><a href="https://github.com/griffithlab/rnaseq_tutorial_wiki.git">Fork it and send a pull request.</a></p>
<p><strong><a href="https://github.com/griffithlab/rnaseq_tutorial/wiki">Table of Contents</a></strong><br>
Module 0: <a class="internal present" href="https://github.com/griffithlab/rnaseq_tutorial/wiki/Authors">Authors</a> | <a class="internal present" href="https://github.com/griffithlab/rnaseq_tutorial/wiki/Citation">Citation</a> | <a class="internal present" href="https://github.com/griffithlab/rnaseq_tutorial/wiki/Syntax">Syntax</a> | <a class="internal present" href="https://github.com/griffithlab/rnaseq_tutorial/wiki/Intro-to-AWS-Cloud-Computing">Intro to AWS</a> | <a class="internal present" href="https://github.com/griffithlab/rnaseq_tutorial/wiki/Logging-into-Amazon-Cloud">Log into AWS</a> | <a class="internal present" href="https://github.com/griffithlab/rnaseq_tutorial/wiki/Unix-Bootcamp">Unix</a>  | <a class="internal present" href="https://github.com/griffithlab/rnaseq_tutorial/wiki/Environment">Environment</a> | <a class="internal present" href="https://github.com/griffithlab/rnaseq_tutorial/wiki/Resources">Resources</a><br>
Module 1: <a class="internal present" href="https://github.com/griffithlab/rnaseq_tutorial/wiki/Installation">Installation</a> | <a class="internal present" href="https://github.com/griffithlab/rnaseq_tutorial/wiki/Reference-Genome">Reference Genomes</a> | <a class="internal present" href="https://github.com/griffithlab/rnaseq_tutorial/wiki/Annotation">Annotations</a> | <a class="internal present" href="https://github.com/griffithlab/rnaseq_tutorial/wiki/Indexing">Indexing</a> | <a class="internal present" href="https://github.com/griffithlab/rnaseq_tutorial/wiki/RNAseq-Data">Data</a> | <a class="internal present" href="https://github.com/griffithlab/rnaseq_tutorial/wiki/PreAlignment-QC">Data QC</a><br>
Module 2: <a class="internal present" href="https://github.com/griffithlab/rnaseq_tutorial/wiki/Adapter-Trim">Adapter Trim</a> | <a class="internal present" href="https://github.com/griffithlab/rnaseq_tutorial/wiki/Alignment">Alignment</a> | <a class="internal present" href="https://github.com/griffithlab/rnaseq_tutorial/wiki/IGV-Tutorial">IGV</a> | <a class="internal present" href="https://github.com/griffithlab/rnaseq_tutorial/wiki/PostAlignment-Visualization">Alignment Visualization</a> | <a class="internal present" href="https://github.com/griffithlab/rnaseq_tutorial/wiki/PostAlignment-QC">Alignment QC</a><br>
Module 3: <a class="internal present" href="https://github.com/griffithlab/rnaseq_tutorial/wiki/Expression">Expression</a> | <a class="internal present" href="https://github.com/griffithlab/rnaseq_tutorial/wiki/Differential-Expression">Differential Expression</a> | <a class="internal present" href="https://github.com/griffithlab/rnaseq_tutorial/wiki/DE-Visualization">DE Visualization</a><br>
Module 4: <a class="internal present" href="https://github.com/griffithlab/rnaseq_tutorial/wiki/Kallisto">Alignment Free - Kallisto</a><br>
Module 5: <a class="internal present" href="https://github.com/griffithlab/rnaseq_tutorial/wiki/Reference-Guided-Transcript-Assembly">Ref Guided</a> | <a class="internal present" href="https://github.com/griffithlab/rnaseq_tutorial/wiki/de-novo-Transcript-Assembly">De novo</a> | <a class="internal present" href="https://github.com/griffithlab/rnaseq_tutorial/wiki/Transcript-Assembly-Merge">Merging</a> | <a class="internal present" href="https://github.com/griffithlab/rnaseq_tutorial/wiki/Differential-Splicing">Differential Splicing</a> | <a class="internal present" href="https://github.com/griffithlab/rnaseq_tutorial/wiki/Transcript-Assembly-Visualization">Splicing Visualization</a><br>
Module 6: <a class="internal present" href="https://github.com/griffithlab/rnaseq_tutorial/wiki/Trinity-Assembly-And-Analysis">Trinity</a><br>
Module 7: <a class="internal present" href="https://github.com/griffithlab/rnaseq_tutorial/wiki/Trinotate-Functional-Annotation">Trinotate</a><br>
Appendix: <a class="internal present" href="https://github.com/griffithlab/rnaseq_tutorial/wiki/Saving-Your-Results">Saving Results</a> | <a class="internal present" href="https://github.com/griffithlab/rnaseq_tutorial/wiki/Abbreviations">Abbreviations</a> | <a class="internal present" href="https://github.com/griffithlab/rnaseq_tutorial/wiki/Lectures">Lectures</a> | <a class="internal present" href="https://github.com/griffithlab/rnaseq_tutorial/wiki/Solutions">Practical Exercise Solutions</a> | <a class="internal present" href="https://github.com/griffithlab/rnaseq_tutorial/wiki/Integrated-Assignment">Integrated Assignment</a> | <a class="internal present" href="https://github.com/griffithlab/rnaseq_tutorial/wiki/Proposed-Improvements">Proposed Improvements</a> | <a class="internal present" href="https://github.com/griffithlab/rnaseq_tutorial/wiki/AWS-Setup">AWS Setup</a><br></p>

          </div>
        </div>
    </div>
    </div>
  </div>
  </div>

  </div>
  <div class="modal-backdrop js-touch-events"></div>
</div>


    </div>
  </div>

  </div>

      
<div class="footer container-lg px-3" role="contentinfo">
  <div class="position-relative d-flex flex-justify-between pt-6 pb-2 mt-6 f6 text-gray border-top border-gray-light ">
    <ul class="list-style-none d-flex flex-wrap ">
      <li class="mr-3">© 2018 <span title="0.28175s from unicorn-1323058819-s55t0">GitHub</span>, Inc.</li>
        <li class="mr-3"><a data-ga-click="Footer, go to terms, text:terms" href="https://github.com/site/terms">Terms</a></li>
        <li class="mr-3"><a data-ga-click="Footer, go to privacy, text:privacy" href="https://github.com/site/privacy">Privacy</a></li>
        <li class="mr-3"><a href="https://help.github.com/articles/github-security/" data-ga-click="Footer, go to security, text:security">Security</a></li>
        <li class="mr-3"><a href="https://status.github.com/" data-ga-click="Footer, go to status, text:status">Status</a></li>
        <li><a data-ga-click="Footer, go to help, text:help" href="https://help.github.com/">Help</a></li>
    </ul>

    <a aria-label="Homepage" title="GitHub" class="footer-octicon" href="https://github.com/">
      <svg height="24" class="octicon octicon-mark-github" viewBox="0 0 16 16" version="1.1" width="24" aria-hidden="true"><path fill-rule="evenodd" d="M8 0C3.58 0 0 3.58 0 8c0 3.54 2.29 6.53 5.47 7.59.4.07.55-.17.55-.38 0-.19-.01-.82-.01-1.49-2.01.37-2.53-.49-2.69-.94-.09-.23-.48-.94-.82-1.13-.28-.15-.68-.52-.01-.53.63-.01 1.08.58 1.23.82.72 1.21 1.87.87 2.33.66.07-.52.28-.87.51-1.07-1.78-.2-3.64-.89-3.64-3.95 0-.87.31-1.59.82-2.15-.08-.2-.36-1.02.08-2.12 0 0 .67-.21 2.2.82.64-.18 1.32-.27 2-.27.68 0 1.36.09 2 .27 1.53-1.04 2.2-.82 2.2-.82.44 1.1.16 1.92.08 2.12.51.56.82 1.27.82 2.15 0 3.07-1.87 3.75-3.65 3.95.29.25.54.73.54 1.48 0 1.07-.01 1.93-.01 2.2 0 .21.15.46.55.38A8.013 8.013 0 0 0 16 8c0-4.42-3.58-8-8-8z"></path></svg>
</a>
   <ul class="list-style-none d-flex flex-wrap ">
        <li class="mr-3"><a data-ga-click="Footer, go to contact, text:contact" href="https://github.com/contact">Contact GitHub</a></li>
      <li class="mr-3"><a href="https://developer.github.com/" data-ga-click="Footer, go to api, text:api">API</a></li>
      <li class="mr-3"><a href="https://training.github.com/" data-ga-click="Footer, go to training, text:training">Training</a></li>
      <li class="mr-3"><a href="https://shop.github.com/" data-ga-click="Footer, go to shop, text:shop">Shop</a></li>
        <li class="mr-3"><a href="https://blog.github.com/" data-ga-click="Footer, go to blog, text:blog">Blog</a></li>
        <li><a data-ga-click="Footer, go to about, text:about" href="https://github.com/about">About</a></li>

    </ul>
  </div>
  <div class="d-flex flex-justify-center pb-6">
    <span class="f6 text-gray-light"></span>
  </div>
</div>



  <div id="ajax-error-message" class="ajax-error-message flash flash-error">
    <svg class="octicon octicon-alert" viewBox="0 0 16 16" version="1.1" width="16" height="16" aria-hidden="true"><path fill-rule="evenodd" d="M8.893 1.5c-.183-.31-.52-.5-.887-.5s-.703.19-.886.5L.138 13.499a.98.98 0 0 0 0 1.001c.193.31.53.501.886.501h13.964c.367 0 .704-.19.877-.5a1.03 1.03 0 0 0 .01-1.002L8.893 1.5zm.133 11.497H6.987v-2.003h2.039v2.003zm0-3.004H6.987V5.987h2.039v4.006z"></path></svg>
    <button type="button" class="flash-close js-ajax-error-dismiss" aria-label="Dismiss error">
      <svg class="octicon octicon-x" viewBox="0 0 12 16" version="1.1" width="12" height="16" aria-hidden="true"><path fill-rule="evenodd" d="M7.48 8l3.75 3.75-1.48 1.48L6 9.48l-3.75 3.75-1.48-1.48L4.52 8 .77 4.25l1.48-1.48L6 6.52l3.75-3.75 1.48 1.48L7.48 8z"></path></svg>
    </button>
    You can’t perform that action at this time.
  </div>


    
    <script crossorigin="anonymous" integrity="sha512-dw6pJdMuGt8qWi9UIwXbz39kSg4xPIVSs8eH9s/wQ7rv3Qv293719SgZR3Ed+8/Au17JplVWepgHKk3XxJHReg==" type="application/javascript" src="./IGV Tutorial · griffithlab_rnaseq_tutorial Wiki_files/frameworks-348dfac1ab5d915b75e0f0c823fd0bbb.js"></script>
    
    <script crossorigin="anonymous" async="async" integrity="sha512-DHTuBUYNLGd7AJONSzgLEp6QdRNcG+9IHGMYXEPZLDostPkRBG23e/t61idVyS35xpsT1boql74nigxxHL7VFQ==" type="application/javascript" src="./IGV Tutorial · griffithlab_rnaseq_tutorial Wiki_files/github-570ef43e73a9c60f604a6f3068ac5436.js"></script>
    
      <script crossorigin="anonymous" integrity="sha512-jh1O+gxnfxubeDtNazfy7e2YjxPEGWp+0q5U9C3MKMkOIVSjD+kv+oNqrVRrxtZ0b7LCpxQTlGdDKkzWW6AYxg==" type="application/javascript" src="./IGV Tutorial · griffithlab_rnaseq_tutorial Wiki_files/wiki-c1ba6d59078a0f68ca32b1c768c82d87.js"></script>

    
  <div class="js-stale-session-flash stale-session-flash flash flash-warn flash-banner d-none">
    <svg class="octicon octicon-alert" viewBox="0 0 16 16" version="1.1" width="16" height="16" aria-hidden="true"><path fill-rule="evenodd" d="M8.893 1.5c-.183-.31-.52-.5-.887-.5s-.703.19-.886.5L.138 13.499a.98.98 0 0 0 0 1.001c.193.31.53.501.886.501h13.964c.367 0 .704-.19.877-.5a1.03 1.03 0 0 0 .01-1.002L8.893 1.5zm.133 11.497H6.987v-2.003h2.039v2.003zm0-3.004H6.987V5.987h2.039v4.006z"></path></svg>
    <span class="signed-in-tab-flash">You signed in with another tab or window. <a href="https://github.com/griffithlab/rnaseq_tutorial/wiki/IGV-Tutorial">Reload</a> to refresh your session.</span>
    <span class="signed-out-tab-flash">You signed out in another tab or window. <a href="https://github.com/griffithlab/rnaseq_tutorial/wiki/IGV-Tutorial">Reload</a> to refresh your session.</span>
  </div>
  <div class="facebox" id="facebox" style="display:none;">
  <div class="facebox-popup">
    <div class="facebox-content" role="dialog" aria-labelledby="facebox-header" aria-describedby="facebox-description">
    </div>
    <button type="button" class="facebox-close js-facebox-close" aria-label="Close modal">
      <svg class="octicon octicon-x" viewBox="0 0 12 16" version="1.1" width="12" height="16" aria-hidden="true"><path fill-rule="evenodd" d="M7.48 8l3.75 3.75-1.48 1.48L6 9.48l-3.75 3.75-1.48-1.48L4.52 8 .77 4.25l1.48-1.48L6 6.52l3.75-3.75 1.48 1.48L7.48 8z"></path></svg>
    </button>
  </div>
</div>

  <div class="Popover js-hovercard-content position-absolute" style="display: none; outline: none;" tabindex="0">
  <div class="Popover-message Popover-message--bottom-left Popover-message--large Box box-shadow-large" style="width:360px;">
  </div>
</div>

<div id="hovercard-aria-description" class="sr-only">
  Press h to open a hovercard with more details.
</div>


  


</body></html>